samples/extract.json

```diff
@@ -320,10 +320,11 @@
                         ],
                         "Theme": {
                             "Code": "ch.Nutzungsplanung",
+                            "SubCode": "ch.NutzungsplanungGrundnutzung",
                             "Text": [
                                 {
                                     "Language": "de",
-                                    "Text": "Nutzungsplanung (kantonal/kommunal)"
+                                    "Text": "Nutzungsplanung (kantonal/kommunal): Grundnutzung"
```

Comment:
Is this what is expected to be a valid JSON Data extract? If: This is not correct. Sub theme titles are not concatenated. The Text in this case would be `Grundnutzung`

You will need to catch the "main Theme" Text via code from the Themes-List in the extract and concatenate it yourself

Reply:
```json
{"$schema": "https://raw.githubusercontent.com/openoereb/schemas/ea4fe8e696b84b923cb9aa9fb27c3ba4e2d8eb5b/extract.json", "GetExtractByIdResponse": {"extract": {"CreationDate": "2022-02-11T09:20:48", "ExtractIdentifier": "5d3380a3-7946-48f2-9f24-6f5939515496", "UpdateDateCS": "2022-02-11T09:20:48", "PLRCadastreAuthority": {"Name": [{"Language": "de", "Text": "\u00d6REB-Katasteraufsichtsbeh\u00f6rde"}], "OfficeAtWeb": [{"Language": "de", "Text": "https://www.cadastre.ch/en/oereb.html"}], "Street": "Seftigenstrasse", "Number": "264", "PostalCode": "3084", "City": "Wabern"}, "RealEstate": {"Type": {"Code": "RealEstate", "Text": [{"Language": "de", "Text": "Liegenschaft"}]}, "Canton": "BL", "MunicipalityName": "Oberwil (BL)", "MunicipalityCode": 2771, "LandRegistryArea": 35121, "PlanForLandRegister": {"ReferenceWMS": [{"Language": "de", "Text": "https://wms.geo.admin.ch/?SERVICE=WMS&REQUEST=GetMap&VERSION=1.3.0&STYLES=default&CRS=EPSG%3A2056&BBOX=2608564.1173571427%2C1261670.8114999998%2C2609194.3826428573%2C1262028.7715&WIDTH=493&HEIGHT=280&FORMAT=image%2Fpng&LAYERS=ch.swisstopo-vd.amtliche-vermessung&SRS=EPSG%3A2056"}], "layerIndex": 0, "layerOpacity": 1.0, "min": {"type": "Point", "coordinates": [2608564.1173571427, 1261670.8114999998], "crs": "EPSG:2056"}, "max": {"type": "Point", "coordinates": [2609194.3826428573, 1262028.7715], "crs": "EPSG:2056"}}, "PlanForLandRegisterMainPage": {"ReferenceWMS": [{"Language": "de", "Text": "https://wms.geo.admin.ch/?SERVICE=WMS&REQUEST=GetMap&VERSION=1.3.0&STYLES=default&CRS=EPSG%3A2056&BBOX=2608564.1173571427%2C1261670.8114999998%2C2609194.3826428573%2C1262028.7715&WIDTH=493&HEIGHT=280&FORMAT=image%2Fpng&LAYERS=ch.swisstopo-vd.amtliche-vermessung&SRS=EPSG%3A2056"}], "layerIndex": 0, "layerOpacity": 1.0, "min": {"type": "Point", "coordinates": [2608564.1173571427, 1261670.8114999998], "crs": "EPSG:2056"}, "max": {"type": "Point", "coordinates": [2609194.3826428573, 1262028.7715], "crs": "EPSG:2056"}}, "Number": "70", "IdentDN": "BL0200002771", "EGRID": "CH113928077734", "RestrictionOnLandownership": [{"LegendText": [{"Language": "de", "Text": "Belastet, weder \u00fcberwachungs- noch sanierungsbed\u00fcrftig"}], "Theme": {"Code": "ch.Nutzungsplanung", "Text": [{"Language": "de", "Text": "Baulinien"}], "SubCode": "ch.Baulinien"}, "Lawstatus": {"Code": "changeWithPreEffect", "Text": [{"Language": "de", "Text": "\u00c4nderung mit Vorwirkung"}]}, "ResponsibleOffice": {"Name": [{"Language": "de", "Text": "Bundesamt f\u00fcr Verkehr BAV"}], "OfficeAtWeb": [{"Language": "de", "Text": "https://www.bav.admin.ch/bav/de/home.html"}]}, "Map": {"ReferenceWMS": [{"Language": "de", "Text": "https://wms.geo.admin.ch/?SERVICE=WMS&REQUEST=GetMap&VERSION=1.3.0&STYLES=default&CRS=EPSG%3A2056&BBOX=2608564.1173571427%2C1261670.8114999998%2C2609194.3826428573%2C1262028.7715&WIDTH=493&HEIGHT=280&FORMAT=image%2Fpng&LAYERS=ch.bav.kataster-belasteter-standorte-oev.oereb&SRS=EPSG%3A2056"}], "OtherLegend": [{"LegendText": [{"Language": "de", "Text": "Belastet, weder \u00fcberwachungs- noch sanierungsbed\u00fcrftig"}], "TypeCode": "StaoTyp3", "TypeCodelist": "https://models.geo.admin.ch/BAFU/KbS_Codetexte_V1_4.xml", "Theme": {"Code": "ch.Nutzungsplanung", "Text": [{"Language": "de", "Text": "Baulinien"}], "SubCode": "ch.Baulinien"}, "SymbolRef": "http://localhost:6543/oereb/image/symbol/ch.Nutzungsplanung/1/StaoTyp3.png?sub_theme_code=ch.Baulinien"}], "layerIndex": 1, "layerOpacity": 0.75, "min": {"type": "Point", "coordinates": [2608564.1173571427, 1261670.8114999998], "crs": "EPSG:2056"}, "max": {"type": "Point", "coordinates": [2609194.3826428573, 1262028.7715], "crs": "EPSG:2056"}}, "SymbolRef": "http://localhost:6543/oereb/image/symbol/ch.Nutzungsplanung/1/StaoTyp2.png?sub_theme_code=ch.Baulinien", "LengthShare": 101, "TypeCode": "StaoTyp2", "TypeCodelist": "https://models.geo.admin.ch/BAFU/KbS_Codetexte_V1_4.xml", "LegalProvisions": [{"Type": {"Code": "Law", "Text": [{"Language": "de", "Text": "Gesetzliche Grundlage"}]}, "Index": 10, "Title": [{"Language": "de", "Text": "Bundesgesetz \u00fcber die Raumplanung"}], "Lawstatus": {"Code": "inForce", "Text": [{"Language": "de", "Text": "Rechtskr\u00e4ftig"}]}, "TextAtWeb": [{"Language": "de", "Text": "https://www.admin.ch/ch/d/sr/c700.html"}], "ResponsibleOffice": {"Name": [{"Language": "de", "Text": "Bundeskanzlei"}], "OfficeAtWeb": [{"Language": "de", "Text": "https://www.bk.admin.ch"}], "UID": "CHE263297189"}, "Abbreviation": [{"Language": "de", "Text": "RPG"}], "OfficialNumber": [{"Language": "de", "Text": "SR 700"}]}, {"Type": {"Code": "Law", "Text": [{"Language": "de", "Text": "Gesetzliche Grundlage"}]}, "Index": 10, "Title": [{"Language": "de", "Text": "Murphys Gesetz"}], "Lawstatus": {"Code": "inForce", "Text": [{"Language": "de", "Text": "Rechtskr\u00e4ftig"}]}, "TextAtWeb": [{"Language": "de", "Text": "https://de.wikipedia.org/wiki/Murphys_Gesetz"}], "ResponsibleOffice": {"Name": [{"Language": "de", "Text": "Rudert-Geoinformatik"}], "OfficeAtWeb": [{"Language": "de", "Text": "https://www.rudert-geoinformatik.ch"}], "UID": "CHE123456778"}, "Abbreviation": [{"Language": "de", "Text": "MG"}], "OfficialNumber": [{"Language": "de", "Text": "SR 969"}], "ArticleNumber": ["1", "2", "3"]}, {"Type": {"Code": "LegalProvision", "Text": [{"Language": "de", "Text": "Rechtsvorschrift"}]}, "Index": 2, "Title": [{"Language": "de", "Text": "B61001"}], "Lawstatus": {"Code": "inForce", "Text": [{"Language": "de", "Text": "Rechtskr\u00e4ftig"}]}, "TextAtWeb": [{"Language": "de", "Text": "https://www.bav.admin.ch/dam/bav/de/dokumente/das-bav/finanzierung/faktenblatt_fernverkehr_konzessionierung.pdf.download.pdf/d_Faktenblatt_Fernverkehr_und_Konzessionierung%20.pdf"}], "ResponsibleOffice": {"Name": [{"Language": "de", "Text": "Bundesamt f\u00fcr Verkehr BAV"}], "OfficeAtWeb": [{"Language": "de", "Text": "https://www.bav.admin.ch/bav/de/home.html"}]}}, {"Type": {"Code": "Law", "Text": [{"Language": "de", "Text": "Gesetzliche Grundlage"}]}, "Index": 1, "Title": [{"Language": "de", "Text": "Umweltschutzgesetz"}], "Lawstatus": {"Code": "inForce", "Text": [{"Language": "de", "Text": "Rechtskr\u00e4ftig"}]}, "TextAtWeb": [{"Language": "de", "Text": "http://www.admin.ch/ch/d/sr/c814_01.html"}], "ResponsibleOffice": {"Name": [{"Language": "de", "Text": "Bundesamt f\u00fcr Verkehr BAV"}], "OfficeAtWeb": [{"Language": "de", "Text": "https://www.bav.admin.ch/bav/de/home.html"}]}, "Abbreviation": [{"Language": "de", "Text": "USG"}], "OfficialNumber": [{"Language": "de", "Text": "SR 814.01"}]}, {"Type": {"Code": "Law", "Text": [{"Language": "de", "Text": "Gesetzliche Grundlage"}]}, "Index": 1, "Title": [{"Language": "de", "Text": "Altlasten-Verordnung"}], "Lawstatus": {"Code": "inForce", "Text": [{"Language": "de", "Text": "Rechtskr\u00e4ftig"}]}, "TextAtWeb": [{"Language": "de", "Text": "http://www.admin.ch/ch/d/sr/c814_680.html"}], "ResponsibleOffice": {"Name": [{"Language": "de", "Text": "Bundesamt f\u00fcr Verkehr BAV"}], "OfficeAtWeb": [{"Language": "de", "Text": "https://www.bav.admin.ch/bav/de/home.html"}]}, "Abbreviation": [{"Language": "de", "Text": "AltlV"}], "OfficialNumber": [{"Language": "de", "Text": "SR 814.680"}]}]}, {"LegendText": [{"Language": "de", "Text": "Belastet, sanierungsbed\u00fcrftig"}], "Theme": {"Code": "ch.Nutzungsplanung", "Text": [{"Language": "de", "Text": "Baulinien"}], "SubCode": "ch.Baulinien"}, "Lawstatus": {"Code": "changeWithoutPreEffect", "Text": [{"Language": "de", "Text": "\u00c4nderung ohne Vorwirkung"}]}, "ResponsibleOffice": {"Name": [{"Language": "de", "Text": "Bundesamt f\u00fcr Verkehr BAV"}], "OfficeAtWeb": [{"Language": "de", "Text": "https://www.bav.admin.ch/bav/de/home.html"}]}, "Map": {"ReferenceWMS": [{"Language": "de", "Text": "https://wms.geo.admin.ch/?SERVICE=WMS&REQUEST=GetMap&VERSION=1.3.0&STYLES=default&CRS=EPSG%3A2056&BBOX=2608564.1173571427%2C1261670.8114999998%2C2609194.3826428573%2C1262028.7715&WIDTH=493&HEIGHT=280&FORMAT=image%2Fpng&LAYERS=ch.bav.kataster-belasteter-standorte-oev.oereb&SRS=EPSG%3A2056"}], "OtherLegend": [{"LegendText": [{"Language": "de", "Text": "Belastet, weder \u00fcberwachungs- noch sanierungsbed\u00fcrftig"}], "TypeCode": "StaoTyp3", "TypeCodelist": "https://models.geo.admin.ch/BAFU/KbS_Codetexte_V1_4.xml", "Theme": {"Code": "ch.Nutzungsplanung", "Text": [{"Language": "de", "Text": "Baulinien"}], "SubCode": "ch.Baulinien"}, "SymbolRef": "http://localhost:6543/oereb/image/symbol/ch.Nutzungsplanung/1/StaoTyp3.png?sub_theme_code=ch.Baulinien"}], "layerIndex": 1, "layerOpacity": 0.75, "min": {"type": "Point", "coordinates": [2608564.1173571427, 1261670.8114999998], "crs": "EPSG:2056"}, "max": {"type": "Point", "coordinates": [2609194.3826428573, 1262028.7715], "crs": "EPSG:2056"}}, "SymbolRef": "http://localhost:6543/oereb/image/symbol/ch.Nutzungsplanung/1/StaoTyp1.png?sub_theme_code=ch.Baulinien", "LengthShare": 63, "TypeCode": "StaoTyp1", "TypeCodelist": "https://models.geo.admin.ch/BAFU/KbS_Codetexte_V1_4.xml", "LegalProvisions": [{"Type": {"Code": "Law", "Text": [{"Language": "de", "Text": "Gesetzliche Grundlage"}]}, "Index": 10, "Title": [{"Language": "de", "Text": "Bundesgesetz \u00fcber die Raumplanung"}], "Lawstatus": {"Code": "inForce", "Text": [{"Language": "de", "Text": "Rechtskr\u00e4ftig"}]}, "TextAtWeb": [{"Language": "de", "Text": "https://www.admin.ch/ch/d/sr/c700.html"}], "ResponsibleOffice": {"Name": [{"Language": "de", "Text": "Bundeskanzlei"}], "OfficeAtWeb": [{"Language": "de", "Text": "https://www.bk.admin.ch"}], "UID": "CHE263297189"}, "Abbreviation": [{"Language": "de", "Text": "RPG"}], "OfficialNumber": [{"Language": "de", "Text": "SR 700"}]}, {"Type": {"Code": "Law", "Text": [{"Language": "de", "Text": "Gesetzliche Grundlage"}]}, "Index": 10, "Title": [{"Language": "de", "Text": "Murphys Gesetz"}], "Lawstatus": {"Code": "inForce", "Text": [{"Language": "de", "Text": "Rechtskr\u00e4ftig"}]}, "TextAtWeb": [{"Language": "de", "Text": "https://de.wikipedia.org/wiki/Murphys_Gesetz"}], "ResponsibleOffice": {"Name": [{"Language": "de", "Text": "Rudert-Geoinformatik"}], "OfficeAtWeb": [{"Language": "de", "Text": "https://www.rudert-geoinformatik.ch"}], "UID": "CHE123456778"}, "Abbreviation": [{"Language": "de", "Text": "MG"}], "OfficialNumber": [{"Language": "de", "Text": "SR 969"}], "ArticleNumber": ["1", "2", "3"]}, {"Type": {"Code": "LegalProvision", "Text": [{"Language": "de", "Text": "Rechtsvorschrift"}]}, "Index": 2, "Title": [{"Language": "de", "Text": "B61001"}], "Lawstatus": {"Code": "inForce", "Text": [{"Language": "de", "Text": "Rechtskr\u00e4ftig"}]}, "TextAtWeb": [{"Language": "de", "Text": "https://www.bav.admin.ch/dam/bav/de/dokumente/das-bav/finanzierung/faktenblatt_fernverkehr_konzessionierung.pdf.download.pdf/d_Faktenblatt_Fernverkehr_und_Konzessionierung%20.pdf"}], "ResponsibleOffice": {"Name": [{"Language": "de", "Text": "Bundesamt f\u00fcr Verkehr BAV"}], "OfficeAtWeb": [{"Language": "de", "Text": "https://www.bav.admin.ch/bav/de/home.html"}]}}, {"Type": {"Code": "Law", "Text": [{"Language": "de", "Text": "Gesetzliche Grundlage"}]}, "Index": 1, "Title": [{"Language": "de", "Text": "Umweltschutzgesetz"}], "Lawstatus": {"Code": "inForce", "Text": [{"Language": "de", "Text": "Rechtskr\u00e4ftig"}]}, "TextAtWeb": [{"Language": "de", "Text": "http://www.admin.ch/ch/d/sr/c814_01.html"}], "ResponsibleOffice": {"Name": [{"Language": "de", "Text": "Bundesamt f\u00fcr Verkehr BAV"}], "OfficeAtWeb": [{"Language": "de", "Text": "https://www.bav.admin.ch/bav/de/home.html"}]}, "Abbreviation": [{"Language": "de", "Text": "USG"}], "OfficialNumber": [{"Language": "de", "Text": "SR 814.01"}]}, {"Type": {"Code": "Law", "Text": [{"Language": "de", "Text": "Gesetzliche Grundlage"}]}, "Index": 1, "Title": [{"Language": "de", "Text": "Altlasten-Verordnung"}], "Lawstatus": {"Code": "inForce", "Text": [{"Language": "de", "Text": "Rechtskr\u00e4ftig"}]}, "TextAtWeb": [{"Language": "de", "Text": "http://www.admin.ch/ch/d/sr/c814_680.html"}], "ResponsibleOffice": {"Name": [{"Language": "de", "Text": "Bundesamt f\u00fcr Verkehr BAV"}], "OfficeAtWeb": [{"Language": "de", "Text": "https://www.bav.admin.ch/bav/de/home.html"}]}, "Abbreviation": [{"Language": "de", "Text": "AltlV"}], "OfficialNumber": [{"Language": "de", "Text": "SR 814.680"}]}]}, {"LegendText": [{"Language": "de", "Text": "Belastet, sanierungsbed\u00fcrftig"}], "Theme": {"Code": "ch.Nutzungsplanung", "Text": [{"Language": "de", "Text": "Baulinien"}], "SubCode": "ch.Baulinien"}, "Lawstatus": {"Code": "inForce", "Text": [{"Language": "de", "Text": "Rechtskr\u00e4ftig"}]}, "ResponsibleOffice": {"Name": [{"Language": "de", "Text": "Bundesamt f\u00fcr Verkehr BAV"}], "OfficeAtWeb": [{"Language": "de", "Text": "https://www.bav.admin.ch/bav/de/home.html"}]}, "Map": {"ReferenceWMS": [{"Language": "de", "Text": "https://wms.geo.admin.ch/?SERVICE=WMS&REQUEST=GetMap&VERSION=1.3.0&STYLES=default&CRS=EPSG%3A2056&BBOX=2608564.1173571427%2C1261670.8114999998%2C2609194.3826428573%2C1262028.7715&WIDTH=493&HEIGHT=280&FORMAT=image%2Fpng&LAYERS=ch.bav.kataster-belasteter-standorte-oev.oereb&SRS=EPSG%3A2056"}], "OtherLegend": [{"LegendText": [{"Language": "de", "Text": "Belastet, weder \u00fcberwachungs- noch sanierungsbed\u00fcrftig"}], "TypeCode": "StaoTyp3", "TypeCodelist": "https://models.geo.admin.ch/BAFU/KbS_Codetexte_V1_4.xml", "Theme": {"Code": "ch.Nutzungsplanung", "Text": [{"Language": "de", "Text": "Baulinien"}], "SubCode": "ch.Baulinien"}, "SymbolRef": "http://localhost:6543/oereb/image/symbol/ch.Nutzungsplanung/1/StaoTyp3.png?sub_theme_code=ch.Baulinien"}], "layerIndex": 1, "layerOpacity": 0.75, "min": {"type": "Point", "coordinates": [2608564.1173571427, 1261670.8114999998], "crs": "EPSG:2056"}, "max": {"type": "Point", "coordinates": [2609194.3826428573, 1262028.7715], "crs": "EPSG:2056"}}, "SymbolRef": "http://localhost:6543/oereb/image/symbol/ch.Nutzungsplanung/1/StaoTyp1.png?sub_theme_code=ch.Baulinien", "NrOfPoints": 2, "TypeCode": "StaoTyp1", "TypeCodelist": "https://models.geo.admin.ch/BAFU/KbS_Codetexte_V1_4.xml", "LegalProvisions": [{"Type": {"Code": "Law", "Text": [{"Language": "de", "Text": "Gesetzliche Grundlage"}]}, "Index": 10, "Title": [{"Language": "de", "Text": "Bundesgesetz \u00fcber die Raumplanung"}], "Lawstatus": {"Code": "inForce", "Text": [{"Language": "de", "Text": "Rechtskr\u00e4ftig"}]}, "TextAtWeb": [{"Language": "de", "Text": "https://www.admin.ch/ch/d/sr/c700.html"}], "ResponsibleOffice": {"Name": [{"Language": "de", "Text": "Bundeskanzlei"}], "OfficeAtWeb": [{"Language": "de", "Text": "https://www.bk.admin.ch"}], "UID": "CHE263297189"}, "Abbreviation": [{"Language": "de", "Text": "RPG"}], "OfficialNumber": [{"Language": "de", "Text": "SR 700"}]}, {"Type": {"Code": "Law", "Text": [{"Language": "de", "Text": "Gesetzliche Grundlage"}]}, "Index": 10, "Title": [{"Language": "de", "Text": "Murphys Gesetz"}], "Lawstatus": {"Code": "inForce", "Text": [{"Language": "de", "Text": "Rechtskr\u00e4ftig"}]}, "TextAtWeb": [{"Language": "de", "Text": "https://de.wikipedia.org/wiki/Murphys_Gesetz"}], "ResponsibleOffice": {"Name": [{"Language": "de", "Text": "Rudert-Geoinformatik"}], "OfficeAtWeb": [{"Language": "de", "Text": "https://www.rudert-geoinformatik.ch"}], "UID": "CHE123456778"}, "Abbreviation": [{"Language": "de", "Text": "MG"}], "OfficialNumber": [{"Language": "de", "Text": "SR 969"}], "ArticleNumber": ["1", "2", "3"]}, {"Type": {"Code": "LegalProvision", "Text": [{"Language": "de", "Text": "Rechtsvorschrift"}]}, "Index": 2, "Title": [{"Language": "de", "Text": "B61001"}], "Lawstatus": {"Code": "inForce", "Text": [{"Language": "de", "Text": "Rechtskr\u00e4ftig"}]}, "TextAtWeb": [{"Language": "de", "Text": "https://www.bav.admin.ch/dam/bav/de/dokumente/das-bav/finanzierung/faktenblatt_fernverkehr_konzessionierung.pdf.download.pdf/d_Faktenblatt_Fernverkehr_und_Konzessionierung%20.pdf"}], "ResponsibleOffice": {"Name": [{"Language": "de", "Text": "Bundesamt f\u00fcr Verkehr BAV"}], "OfficeAtWeb": [{"Language": "de", "Text": "https://www.bav.admin.ch/bav/de/home.html"}]}}, {"Type": {"Code": "Law", "Text": [{"Language": "de", "Text": "Gesetzliche Grundlage"}]}, "Index": 1, "Title": [{"Language": "de", "Text": "Umweltschutzgesetz"}], "Lawstatus": {"Code": "inForce", "Text": [{"Language": "de", "Text": "Rechtskr\u00e4ftig"}]}, "TextAtWeb": [{"Language": "de", "Text": "http://www.admin.ch/ch/d/sr/c814_01.html"}], "ResponsibleOffice": {"Name": [{"Language": "de", "Text": "Bundesamt f\u00fcr Verkehr BAV"}], "OfficeAtWeb": [{"Language": "de", "Text": "https://www.bav.admin.ch/bav/de/home.html"}]}, "Abbreviation": [{"Language": "de", "Text": "USG"}], "OfficialNumber": [{"Language": "de", "Text": "SR 814.01"}]}, {"Type": {"Code": "Law", "Text": [{"Language": "de", "Text": "Gesetzliche Grundlage"}]}, "Index": 1, "Title": [{"Language": "de", "Text": "Altlasten-Verordnung"}], "Lawstatus": {"Code": "inForce", "Text": [{"Language": "de", "Text": "Rechtskr\u00e4ftig"}]}, "TextAtWeb": [{"Language": "de", "Text": "http://www.admin.ch/ch/d/sr/c814_680.html"}], "ResponsibleOffice": {"Name": [{"Language": "de", "Text": "Bundesamt f\u00fcr Verkehr BAV"}], "OfficeAtWeb": [{"Language": "de", "Text": "https://www.bav.admin.ch/bav/de/home.html"}]}, "Abbreviation": [{"Language": "de", "Text": "AltlV"}], "OfficialNumber": [{"Language": "de", "Text": "SR 814.680"}]}]}, {"LegendText": [{"Language": "de", "Text": "Belastet, untersuchungsbed\u00fcrftig"}], "Theme": {"Code": "ch.Nutzungsplanung", "Text": [{"Language": "de", "Text": "SubTheme"}], "SubCode": "ch.SubTheme"}, "Lawstatus": {"Code": "changeWithPreEffect", "Text": [{"Language": "de", "Text": "\u00c4nderung mit Vorwirkung"}]}, "ResponsibleOffice": {"Name": [{"Language": "de", "Text": "Bundesamt f\u00fcr Verkehr BAV"}], "OfficeAtWeb": [{"Language": "de", "Text": "https://www.bav.admin.ch/bav/de/home.html"}]}, "Map": {"ReferenceWMS": [{"Language": "de", "Text": "https://wms.geo.admin.ch/?SERVICE=WMS&REQUEST=GetMap&VERSION=1.3.0&STYLES=default&CRS=EPSG%3A2056&BBOX=2608564.1173571427%2C1261670.8114999998%2C2609194.3826428573%2C1262028.7715&WIDTH=493&HEIGHT=280&FORMAT=image%2Fpng&LAYERS=ch.bav.kataster-belasteter-standorte-oev.oereb&SRS=EPSG%3A2056"}], "OtherLegend": [{"LegendText": [{"Language": "de", "Text": "Belastet, weder \u00fcberwachungs- noch sanierungsbed\u00fcrftig"}], "TypeCode": "StaoTyp3", "TypeCodelist": "https://models.geo.admin.ch/BAFU/KbS_Codetexte_V1_4.xml", "Theme": {"Code": "ch.Nutzungsplanung", "Text": [{"Language": "de", "Text": "Baulinien"}], "SubCode": "ch.Baulinien"}, "SymbolRef": "http://localhost:6543/oereb/image/symbol/ch.Nutzungsplanung/1/StaoTyp3.png?sub_theme_code=ch.Baulinien"}], "layerIndex": 1, "layerOpacity": 0.75, "min": {"type": "Point", "coordinates": [2608564.1173571427, 1261670.8114999998], "crs": "EPSG:2056"}, "max": {"type": "Point", "coordinates": [2609194.3826428573, 1262028.7715], "crs": "EPSG:2056"}}, "SymbolRef": "http://localhost:6543/oereb/image/symbol/ch.Nutzungsplanung/1/StaoTyp2.png?sub_theme_code=ch.SubTheme", "AreaShare": 7824, "TypeCode": "StaoTyp2", "TypeCodelist": "https://models.geo.admin.ch/BAFU/KbS_Codetexte_V1_4.xml", "PartInPercent": 22.3, "LegalProvisions": [{"Type": {"Code": "Law", "Text": [{"Language": "de", "Text": "Gesetzliche Grundlage"}]}, "Index": 10, "Title": [{"Language": "de", "Text": "Bundesgesetz \u00fcber die Raumplanung"}], "Lawstatus": {"Code": "inForce", "Text": [{"Language": "de", "Text": "Rechtskr\u00e4ftig"}]}, "TextAtWeb": [{"Language": "de", "Text": "https://www.admin.ch/ch/d/sr/c700.html"}], "ResponsibleOffice": {"Name": [{"Language": "de", "Text": "Bundeskanzlei"}], "OfficeAtWeb": [{"Language": "de", "Text": "https://www.bk.admin.ch"}], "UID": "CHE263297189"}, "Abbreviation": [{"Language": "de", "Text": "RPG"}], "OfficialNumber": [{"Language": "de", "Text": "SR 700"}]}, {"Type": {"Code": "Law", "Text": [{"Language": "de", "Text": "Gesetzliche Grundlage"}]}, "Index": 10, "Title": [{"Language": "de", "Text": "Murphys Gesetz"}], "Lawstatus": {"Code": "inForce", "Text": [{"Language": "de", "Text": "Rechtskr\u00e4ftig"}]}, "TextAtWeb": [{"Language": "de", "Text": "https://de.wikipedia.org/wiki/Murphys_Gesetz"}], "ResponsibleOffice": {"Name": [{"Language": "de", "Text": "Rudert-Geoinformatik"}], "OfficeAtWeb": [{"Language": "de", "Text": "https://www.rudert-geoinformatik.ch"}], "UID": "CHE123456778"}, "Abbreviation": [{"Language": "de", "Text": "MG"}], "OfficialNumber": [{"Language": "de", "Text": "SR 969"}], "ArticleNumber": ["1", "2", "3"]}, {"Type": {"Code": "LegalProvision", "Text": [{"Language": "de", "Text": "Rechtsvorschrift"}]}, "Index": 2, "Title": [{"Language": "de", "Text": "Revision Ortsplanung"}], "Lawstatus": {"Code": "inForce", "Text": [{"Language": "de", "Text": "Rechtskr\u00e4ftig"}]}, "TextAtWeb": [{"Language": "de", "Text": "https://oereb-gr-preview.000.ch/api/attachments/214"}], "ResponsibleOffice": {"Name": [{"Language": "de", "Text": "Bundesamt f\u00fcr Verkehr BAV"}], "OfficeAtWeb": [{"Language": "de", "Text": "https://www.bav.admin.ch/bav/de/home.html"}]}, "OfficialNumber": [{"Language": "de", "Text": "07.447"}]}, {"Type": {"Code": "Law", "Text": [{"Language": "de", "Text": "Gesetzliche Grundlage"}]}, "Index": 1, "Title": [{"Language": "de", "Text": "Umweltschutzgesetz"}], "Lawstatus": {"Code": "inForce", "Text": [{"Language": "de", "Text": "Rechtskr\u00e4ftig"}]}, "TextAtWeb": [{"Language": "de", "Text": "http://www.admin.ch/ch/d/sr/c814_01.html"}], "ResponsibleOffice": {"Name": [{"Language": "de", "Text": "Bundesamt f\u00fcr Verkehr BAV"}], "OfficeAtWeb": [{"Language": "de", "Text": "https://www.bav.admin.ch/bav/de/home.html"}]}, "Abbreviation": [{"Language": "de", "Text": "USG"}], "OfficialNumber": [{"Language": "de", "Text": "SR 814.01"}]}, {"Type": {"Code": "LegalProvision", "Text": [{"Language": "de", "Text": "Rechtsvorschrift"}]}, "Index": 2, "Title": [{"Language": "de", "Text": "B61001"}], "Lawstatus": {"Code": "inForce", "Text": [{"Language": "de", "Text": "Rechtskr\u00e4ftig"}]}, "TextAtWeb": [{"Language": "de", "Text": "https://www.bav.admin.ch/dam/bav/de/dokumente/das-bav/finanzierung/faktenblatt_fernverkehr_konzessionierung.pdf.download.pdf/d_Faktenblatt_Fernverkehr_und_Konzessionierung%20.pdf"}], "ResponsibleOffice": {"Name": [{"Language": "de", "Text": "Bundesamt f\u00fcr Verkehr BAV"}], "OfficeAtWeb": [{"Language": "de", "Text": "https://www.bav.admin.ch/bav/de/home.html"}]}}, {"Type": {"Code": "Law", "Text": [{"Language": "de", "Text": "Gesetzliche Grundlage"}]}, "Index": 1, "Title": [{"Language": "de", "Text": "Altlasten-Verordnung"}], "Lawstatus": {"Code": "inForce", "Text": [{"Language": "de", "Text": "Rechtskr\u00e4ftig"}]}, "TextAtWeb": [{"Language": "de", "Text": "http://www.admin.ch/ch/d/sr/c814_680.html"}], "ResponsibleOffice": {"Name": [{"Language": "de", "Text": "Bundesamt f\u00fcr Verkehr BAV"}], "OfficeAtWeb": [{"Language": "de", "Text": "https://www.bav.admin.ch/bav/de/home.html"}]}, "Abbreviation": [{"Language": "de", "Text": "AltlV"}], "OfficialNumber": [{"Language": "de", "Text": "SR 814.680"}]}, {"Type": {"Code": "LegalProvision", "Text": [{"Language": "de", "Text": "Rechtsvorschrift"}]}, "Index": 2, "Title": [{"Language": "de", "Text": "Revision Ortsplanung"}], "Lawstatus": {"Code": "inForce", "Text": [{"Language": "de", "Text": "Rechtskr\u00e4ftig"}]}, "TextAtWeb": [{"Language": "de", "Text": "https://oereb-gr-preview.000.ch/api/attachments/213"}], "ResponsibleOffice": {"Name": [{"Language": "de", "Text": "Bundesamt f\u00fcr Verkehr BAV"}], "OfficeAtWeb": [{"Language": "de", "Text": "https://www.bav.admin.ch/bav/de/home.html"}]}, "OfficialNumber": [{"Language": "de", "Text": "07.447"}]}, {"Type": {"Code": "Law", "Text": [{"Language": "de", "Text": "Gesetzliche Grundlage"}]}, "Index": 1, "Title": [{"Language": "de", "Text": "Umweltschutzgesetz"}], "Lawstatus": {"Code": "inForce", "Text": [{"Language": "de", "Text": "Rechtskr\u00e4ftig"}]}, "TextAtWeb": [{"Language": "de", "Text": "http://www.admin.ch/ch/d/sr/c814_01.html"}], "ResponsibleOffice": {"Name": [{"Language": "de", "Text": "Bundesamt f\u00fcr Verkehr BAV"}], "OfficeAtWeb": [{"Language": "de", "Text": "https://www.bav.admin.ch/bav/de/home.html"}]}, "Abbreviation": [{"Language": "de", "Text": "USG"}], "OfficialNumber": [{"Language": "de", "Text": "SR 814.01"}]}, {"Type": {"Code": "Law", "Text": [{"Language": "de", "Text": "Gesetzliche Grundlage"}]}, "Index": 1, "Title": [{"Language": "de", "Text": "Bundesgesetz \u00fcber die Raumplanung (Raumplanungsgesetz, RPG)"}], "Lawstatus": {"Code": "inForce", "Text": [{"Language": "de", "Text": "Rechtskr\u00e4ftig"}]}, "TextAtWeb": [{"Language": "de", "Text": "http://www.lexfind.ch/dtah/167348/2"}], "ResponsibleOffice": {"Name": [{"Language": "de", "Text": "Bundesamt f\u00fcr Verkehr BAV"}], "OfficeAtWeb": [{"Language": "de", "Text": "https://www.bav.admin.ch/bav/de/home.html"}]}, "Abbreviation": [{"Language": "de", "Text": "AltlV"}], "OfficialNumber": [{"Language": "de", "Text": "SR 700"}]}, {"Type": {"Code": "Law", "Text": [{"Language": "de", "Text": "Gesetzliche Grundlage"}]}, "Index": 1, "Title": [{"Language": "de", "Text": "Umweltschutzgesetz"}], "Lawstatus": {"Code": "inForce", "Text": [{"Language": "de", "Text": "Rechtskr\u00e4ftig"}]}, "TextAtWeb": [{"Language": "de", "Text": "http://www.admin.ch/ch/d/sr/c814_01.html"}], "ResponsibleOffice": {"Name": [{"Language": "de", "Text": "Bundesamt f\u00fcr Verkehr BAV"}], "OfficeAtWeb": [{"Language": "de", "Text": "https://www.bav.admin.ch/bav/de/home.html"}]}, "Abbreviation": [{"Language": "de", "Text": "USG"}], "OfficialNumber": [{"Language": "de", "Text": "SR 814.01"}]}]}, {"LegendText": [{"Language": "de", "Text": "Belastet, weder \u00fcberwachungs- noch sanierungsbed\u00fcrftig"}], "Theme": {"Code": "ch.Nutzungsplanung", "Text": [{"Language": "de", "Text": "SubTheme"}], "SubCode": "ch.SubTheme"}, "Lawstatus": {"Code": "inForce", "Text": [{"Language": "de", "Text": "Rechtskr\u00e4ftig"}]}, "ResponsibleOffice": {"Name": [{"Language": "de", "Text": "Bundesamt f\u00fcr Verkehr BAV"}], "OfficeAtWeb": [{"Language": "de", "Text": "https://www.bav.admin.ch/bav/de/home.html"}]}, "Map": {"ReferenceWMS": [{"Language": "de", "Text": "https://wms.geo.admin.ch/?SERVICE=WMS&REQUEST=GetMap&VERSION=1.3.0&STYLES=default&CRS=EPSG%3A2056&BBOX=2608564.1173571427%2C1261670.8114999998%2C2609194.3826428573%2C1262028.7715&WIDTH=493&HEIGHT=280&FORMAT=image%2Fpng&LAYERS=ch.bav.kataster-belasteter-standorte-oev.oereb&SRS=EPSG%3A2056"}], "OtherLegend": [{"LegendText": [{"Language": "de", "Text": "Belastet, weder \u00fcberwachungs- noch sanierungsbed\u00fcrftig"}], "TypeCode": "StaoTyp3", "TypeCodelist": "https://models.geo.admin.ch/BAFU/KbS_Codetexte_V1_4.xml", "Theme": {"Code": "ch.Nutzungsplanung", "Text": [{"Language": "de", "Text": "Baulinien"}], "SubCode": "ch.Baulinien"}, "SymbolRef": "http://localhost:6543/oereb/image/symbol/ch.Nutzungsplanung/1/StaoTyp3.png?sub_theme_code=ch.Baulinien"}], "layerIndex": 1, "layerOpacity": 0.75, "min": {"type": "Point", "coordinates": [2608564.1173571427, 1261670.8114999998], "crs": "EPSG:2056"}, "max": {"type": "Point", "coordinates": [2609194.3826428573, 1262028.7715], "crs": "EPSG:2056"}}, "SymbolRef": "http://localhost:6543/oereb/image/symbol/ch.Nutzungsplanung/1/StaoTyp1.png?sub_theme_code=ch.SubTheme", "AreaShare": 3608, "TypeCode": "StaoTyp1", "TypeCodelist": "https://models.geo.admin.ch/BAFU/KbS_Codetexte_V1_4.xml", "PartInPercent": 10.3, "LegalProvisions": [{"Type": {"Code": "Law", "Text": [{"Language": "de", "Text": "Gesetzliche Grundlage"}]}, "Index": 10, "Title": [{"Language": "de", "Text": "Bundesgesetz \u00fcber die Raumplanung"}], "Lawstatus": {"Code": "inForce", "Text": [{"Language": "de", "Text": "Rechtskr\u00e4ftig"}]}, "TextAtWeb": [{"Language": "de", "Text": "https://www.admin.ch/ch/d/sr/c700.html"}], "ResponsibleOffice": {"Name": [{"Language": "de", "Text": "Bundeskanzlei"}], "OfficeAtWeb": [{"Language": "de", "Text": "https://www.bk.admin.ch"}], "UID": "CHE263297189"}, "Abbreviation": [{"Language": "de", "Text": "RPG"}], "OfficialNumber": [{"Language": "de", "Text": "SR 700"}]}, {"Type": {"Code": "Law", "Text": [{"Language": "de", "Text": "Gesetzliche Grundlage"}]}, "Index": 10, "Title": [{"Language": "de", "Text": "Murphys Gesetz"}], "Lawstatus": {"Code": "inForce", "Text": [{"Language": "de", "Text": "Rechtskr\u00e4ftig"}]}, "TextAtWeb": [{"Language": "de", "Text": "https://de.wikipedia.org/wiki/Murphys_Gesetz"}], "ResponsibleOffice": {"Name": [{"Language": "de", "Text": "Rudert-Geoinformatik"}], "OfficeAtWeb": [{"Language": "de", "Text": "https://www.rudert-geoinformatik.ch"}], "UID": "CHE123456778"}, "Abbreviation": [{"Language": "de", "Text": "MG"}], "OfficialNumber": [{"Language": "de", "Text": "SR 969"}], "ArticleNumber": ["1", "2", "3"]}, {"Type": {"Code": "LegalProvision", "Text": [{"Language": "de", "Text": "Rechtsvorschrift"}]}, "Index": 2, "Title": [{"Language": "de", "Text": "B61001"}], "Lawstatus": {"Code": "inForce", "Text": [{"Language": "de", "Text": "Rechtskr\u00e4ftig"}]}, "TextAtWeb": [{"Language": "de", "Text": "https://www.bav.admin.ch/dam/bav/de/dokumente/das-bav/finanzierung/faktenblatt_fernverkehr_konzessionierung.pdf.download.pdf/d_Faktenblatt_Fernverkehr_und_Konzessionierung%20.pdf"}], "ResponsibleOffice": {"Name": [{"Language": "de", "Text": "Bundesamt f\u00fcr Verkehr BAV"}], "OfficeAtWeb": [{"Language": "de", "Text": "https://www.bav.admin.ch/bav/de/home.html"}]}}, {"Type": {"Code": "Law", "Text": [{"Language": "de", "Text": "Gesetzliche Grundlage"}]}, "Index": 1, "Title": [{"Language": "de", "Text": "Umweltschutzgesetz"}], "Lawstatus": {"Code": "inForce", "Text": [{"Language": "de", "Text": "Rechtskr\u00e4ftig"}]}, "TextAtWeb": [{"Language": "de", "Text": "http://www.admin.ch/ch/d/sr/c814_01.html"}], "ResponsibleOffice": {"Name": [{"Language": "de", "Text": "Bundesamt f\u00fcr Verkehr BAV"}], "OfficeAtWeb": [{"Language": "de", "Text": "https://www.bav.admin.ch/bav/de/home.html"}]}, "Abbreviation": [{"Language": "de", "Text": "USG"}], "OfficialNumber": [{"Language": "de", "Text": "SR 814.01"}]}, {"Type": {"Code": "Law", "Text": [{"Language": "de", "Text": "Gesetzliche Grundlage"}]}, "Index": 1, "Title": [{"Language": "de", "Text": "Altlasten-Verordnung"}], "Lawstatus": {"Code": "inForce", "Text": [{"Language": "de", "Text": "Rechtskr\u00e4ftig"}]}, "TextAtWeb": [{"Language": "de", "Text": "http://www.admin.ch/ch/d/sr/c814_680.html"}], "ResponsibleOffice": {"Name": [{"Language": "de", "Text": "Bundesamt f\u00fcr Verkehr BAV"}], "OfficeAtWeb": [{"Language": "de", "Text": "https://www.bav.admin.ch/bav/de/home.html"}]}, "Abbreviation": [{"Language": "de", "Text": "AltlV"}], "OfficialNumber": [{"Language": "de", "Text": "SR 814.680"}]}]}, {"LegendText": [{"Language": "de", "Text": "Belastet, keine sch\u00e4dlichen oder l\u00e4stigen Einwirkungen zu erwarten"}], "Theme": {"Code": "ch.BelasteteStandorteZivileFlugplaetze", "Text": [{"Language": "de", "Text": "Kataster der belasteten Standorte im Bereich der zivilen Flugpl\u00e4tze"}]}, "Lawstatus": {"Code": "inForce", "Text": [{"Language": "de", "Text": "Rechtskr\u00e4ftig"}]}, "ResponsibleOffice": {"Name": [{"Language": "de", "Text": "Bundesamt f\u00fcr Zivilluftfahrt BAZL"}], "OfficeAtWeb": [{"Language": "de", "Text": "https://www.bazl.admin.ch/bazl/de/home/fachleute/flugplaetze/kataster-der-belasteten-standorte-auf-zivilflugplaetzen--kbs-baz.html"}], "UID": "CHE188858471"}, "Map": {"ReferenceWMS": [{"Language": "de", "Text": "https://wms.geo.admin.ch/?SERVICE=WMS&REQUEST=GetMap&VERSION=1.3.0&LAYERS=ch.bazl.kataster-belasteter-standorte-zivilflugplaetze_v2_0.oereb&STYLES=default&CRS=EPSG%3A2056&BBOX=2608564.1173571427%2C1261670.8114999998%2C2609194.3826428573%2C1262028.7715&WIDTH=493&HEIGHT=280&FORMAT=image%2Fpng&LANG=de&SRS=EPSG%3A2056"}], "layerIndex": 1, "layerOpacity": 0.75, "min": {"type": "Point", "coordinates": [2608564.1173571427, 1261670.8114999998], "crs": "EPSG:2056"}, "max": {"type": "Point", "coordinates": [2609194.3826428573, 1262028.7715], "crs": "EPSG:2056"}}, "SymbolRef": "http://localhost:6543/oereb/image/symbol/ch.BelasteteStandorteZivileFlugplaetze/87/StatusAltlV1.png", "NrOfPoints": 1, "TypeCode": "StatusAltlV1", "TypeCodelist": "https://models.geo.admin.ch/BAFU/KbS_Codetexte_V1_5.xml", "LegalProvisions": [{"Type": {"Code": "Law", "Text": [{"Language": "de", "Text": "Gesetzliche Grundlage"}]}, "Index": 410, "Title": [{"Language": "de", "Text": "Bundesgesetz \u00fcber den Umweltschutz"}], "Lawstatus": {"Code": "inForce", "Text": [{"Language": "de", "Text": "Rechtskr\u00e4ftig"}]}, "TextAtWeb": [{"Language": "de", "Text": "https://www.admin.ch/ch/d/sr/c814_01.html"}], "ResponsibleOffice": {"Name": [{"Language": "de", "Text": "Bundeskanzlei"}], "OfficeAtWeb": [{"Language": "de", "Text": "https://www.bk.admin.ch"}], "UID": "CHE263297189"}, "Abbreviation": [{"Language": "de", "Text": "USG"}], "OfficialNumber": [{"Language": "de", "Text": "SR 814.01"}]}, {"Type": {"Code": "Law", "Text": [{"Language": "de", "Text": "Gesetzliche Grundlage"}]}, "Index": 420, "Title": [{"Language": "de", "Text": "Verordnung \u00fcber die Sanierung von belasteten Standorten"}], "Lawstatus": {"Code": "inForce", "Text": [{"Language": "de", "Text": "Rechtskr\u00e4ftig"}]}, "TextAtWeb": [{"Language": "de", "Text": "https://www.admin.ch/ch/d/sr/c814_680.html"}], "ResponsibleOffice": {"Name": [{"Language": "de", "Text": "Bundeskanzlei"}], "OfficeAtWeb": [{"Language": "de", "Text": "https://www.bk.admin.ch"}], "UID": "CHE263297189"}, "Abbreviation": [{"Language": "de", "Text": "AltlV"}], "OfficialNumber": [{"Language": "de", "Text": "SR 814.680"}]}, {"Type": {"Code": "LegalProvision", "Text": [{"Language": "de", "Text": "Rechtsvorschrift"}]}, "Index": 1, "Title": [{"Language": "de", "Text": "BL-Ditt-3-I-01"}], "Lawstatus": {"Code": "inForce", "Text": [{"Language": "de", "Text": "Rechtskr\u00e4ftig"}]}, "TextAtWeb": [{"Language": "de", "Text": "https://map.geo.admin.ch/?zoom=12&bgLayer=ch.swisstopo.swissimage&layers=ch.bazl.kataster-belasteter-standorte-zivilflugplaetze&layers_opacity=0.75&layers_visibility=true&Y=2603990&X=1254150&lang=de"}], "ResponsibleOffice": {"Name": [{"Language": "de", "Text": "Bundesamt f\u00fcr Zivilluftfahrt BAZL"}], "OfficeAtWeb": [{"Language": "de", "Text": "https://www.bazl.admin.ch/bazl/de/home/fachleute/flugplaetze/kataster-der-belasteten-standorte-auf-zivilflugplaetzen--kbs-baz.html"}], "UID": "CHE188858471"}}]}, {"LegendText": [{"Language": "de", "Text": "Belastet, untersuchungsbed\u00fcrftig"}], "Theme": {"Code": "ch.BelasteteStandorteOeffentlicherVerkehr", "Text": [{"Language": "de", "Text": "Kataster der belasteten Standorte im Bereich des \u00f6ffentlichen Verkehrs"}]}, "Lawstatus": {"Code": "changeWithPreEffect", "Text": [{"Language": "de", "Text": "\u00c4nderung mit Vorwirkung"}]}, "ResponsibleOffice": {"Name": [{"Language": "de", "Text": "Bundesamt f\u00fcr Verkehr BAV"}], "OfficeAtWeb": [{"Language": "de", "Text": "https://www.bav.admin.ch/bav/de/home.html"}]}, "Map": {"ReferenceWMS": [{"Language": "de", "Text": "https://wms.geo.admin.ch/?SERVICE=WMS&REQUEST=GetMap&VERSION=1.3.0&STYLES=default&CRS=EPSG%3A2056&BBOX=2608564.1173571427%2C1261670.8114999998%2C2609194.3826428573%2C1262028.7715&WIDTH=493&HEIGHT=280&FORMAT=image%2Fpng&LAYERS=ch.bav.kataster-belasteter-standorte-oev.oereb&SRS=EPSG%3A2056"}], "layerIndex": 1, "layerOpacity": 0.75, "min": {"type": "Point", "coordinates": [2608564.1173571427, 1261670.8114999998], "crs": "EPSG:2056"}, "max": {"type": "Point", "coordinates": [2609194.3826428573, 1262028.7715], "crs": "EPSG:2056"}}, "SymbolRef": "http://localhost:6543/oereb/image/symbol/ch.BelasteteStandorteOeffentlicherVerkehr/1/StaoTyp2.png", "AreaShare": 7824, "TypeCode": "StaoTyp2", "TypeCodelist": "https://models.geo.admin.ch/BAFU/KbS_Codetexte_V1_4.xml", "PartInPercent": 22.3, "LegalProvisions": [{"Type": {"Code": "Law", "Text": [{"Language": "de", "Text": "Gesetzliche Grundlage"}]}, "Index": 410, "Title": [{"Language": "de", "Text": "Bundesgesetz \u00fcber den Umweltschutz"}], "Lawstatus": {"Code": "inForce", "Text": [{"Language": "de", "Text": "Rechtskr\u00e4ftig"}]}, "TextAtWeb": [{"Language": "de", "Text": "https://www.admin.ch/ch/d/sr/c814_01.html"}], "ResponsibleOffice": {"Name": [{"Language": "de", "Text": "Bundeskanzlei"}], "OfficeAtWeb": [{"Language": "de", "Text": "https://www.bk.admin.ch"}], "UID": "CHE263297189"}, "Abbreviation": [{"Language": "de", "Text": "USG"}], "OfficialNumber": [{"Language": "de", "Text": "SR 814.01"}]}, {"Type": {"Code": "Law", "Text": [{"Language": "de", "Text": "Gesetzliche Grundlage"}]}, "Index": 420, "Title": [{"Language": "de", "Text": "Verordnung \u00fcber die Sanierung von belasteten Standorten"}], "Lawstatus": {"Code": "inForce", "Text": [{"Language": "de", "Text": "Rechtskr\u00e4ftig"}]}, "TextAtWeb": [{"Language": "de", "Text": "https://www.admin.ch/ch/d/sr/c814_680.html"}], "ResponsibleOffice": {"Name": [{"Language": "de", "Text": "Bundeskanzlei"}], "OfficeAtWeb": [{"Language": "de", "Text": "https://www.bk.admin.ch"}], "UID": "CHE263297189"}, "Abbreviation": [{"Language": "de", "Text": "AltlV"}], "OfficialNumber": [{"Language": "de", "Text": "SR 814.680"}]}, {"Type": {"Code": "LegalProvision", "Text": [{"Language": "de", "Text": "Rechtsvorschrift"}]}, "Index": 2, "Title": [{"Language": "de", "Text": "Revision Ortsplanung"}], "Lawstatus": {"Code": "inForce", "Text": [{"Language": "de", "Text": "Rechtskr\u00e4ftig"}]}, "TextAtWeb": [{"Language": "de", "Text": "https://oereb-gr-preview.000.ch/api/attachments/214"}], "ResponsibleOffice": {"Name": [{"Language": "de", "Text": "Bundesamt f\u00fcr Verkehr BAV"}], "OfficeAtWeb": [{"Language": "de", "Text": "https://www.bav.admin.ch/bav/de/home.html"}]}, "OfficialNumber": [{"Language": "de", "Text": "07.447"}]}, {"Type": {"Code": "Law", "Text": [{"Language": "de", "Text": "Gesetzliche Grundlage"}]}, "Index": 1, "Title": [{"Language": "de", "Text": "Umweltschutzgesetz"}], "Lawstatus": {"Code": "inForce", "Text": [{"Language": "de", "Text": "Rechtskr\u00e4ftig"}]}, "TextAtWeb": [{"Language": "de", "Text": "http://www.admin.ch/ch/d/sr/c814_01.html"}], "ResponsibleOffice": {"Name": [{"Language": "de", "Text": "Bundesamt f\u00fcr Verkehr BAV"}], "OfficeAtWeb": [{"Language": "de", "Text": "https://www.bav.admin.ch/bav/de/home.html"}]}, "Abbreviation": [{"Language": "de", "Text": "USG"}], "OfficialNumber": [{"Language": "de", "Text": "SR 814.01"}]}, {"Type": {"Code": "LegalProvision", "Text": [{"Language": "de", "Text": "Rechtsvorschrift"}]}, "Index": 2, "Title": [{"Language": "de", "Text": "B61001"}], "Lawstatus": {"Code": "inForce", "Text": [{"Language": "de", "Text": "Rechtskr\u00e4ftig"}]}, "TextAtWeb": [{"Language": "de", "Text": "https://www.bav.admin.ch/dam/bav/de/dokumente/das-bav/finanzierung/faktenblatt_fernverkehr_konzessionierung.pdf.download.pdf/d_Faktenblatt_Fernverkehr_und_Konzessionierung%20.pdf"}], "ResponsibleOffice": {"Name": [{"Language": "de", "Text": "Bundesamt f\u00fcr Verkehr BAV"}], "OfficeAtWeb": [{"Language": "de", "Text": "https://www.bav.admin.ch/bav/de/home.html"}]}}, {"Type": {"Code": "Law", "Text": [{"Language": "de", "Text": "Gesetzliche Grundlage"}]}, "Index": 1, "Title": [{"Language": "de", "Text": "Altlasten-Verordnung"}], "Lawstatus": {"Code": "inForce", "Text": [{"Language": "de", "Text": "Rechtskr\u00e4ftig"}]}, "TextAtWeb": [{"Language": "de", "Text": "http://www.admin.ch/ch/d/sr/c814_680.html"}], "ResponsibleOffice": {"Name": [{"Language": "de", "Text": "Bundesamt f\u00fcr Verkehr BAV"}], "OfficeAtWeb": [{"Language": "de", "Text": "https://www.bav.admin.ch/bav/de/home.html"}]}, "Abbreviation": [{"Language": "de", "Text": "AltlV"}], "OfficialNumber": [{"Language": "de", "Text": "SR 814.680"}]}, {"Type": {"Code": "LegalProvision", "Text": [{"Language": "de", "Text": "Rechtsvorschrift"}]}, "Index": 2, "Title": [{"Language": "de", "Text": "Revision Ortsplanung"}], "Lawstatus": {"Code": "inForce", "Text": [{"Language": "de", "Text": "Rechtskr\u00e4ftig"}]}, "TextAtWeb": [{"Language": "de", "Text": "https://oereb-gr-preview.000.ch/api/attachments/213"}], "ResponsibleOffice": {"Name": [{"Language": "de", "Text": "Bundesamt f\u00fcr Verkehr BAV"}], "OfficeAtWeb": [{"Language": "de", "Text": "https://www.bav.admin.ch/bav/de/home.html"}]}, "OfficialNumber": [{"Language": "de", "Text": "07.447"}]}, {"Type": {"Code": "Law", "Text": [{"Language": "de", "Text": "Gesetzliche Grundlage"}]}, "Index": 1, "Title": [{"Language": "de", "Text": "Umweltschutzgesetz"}], "Lawstatus": {"Code": "inForce", "Text": [{"Language": "de", "Text": "Rechtskr\u00e4ftig"}]}, "TextAtWeb": [{"Language": "de", "Text": "http://www.admin.ch/ch/d/sr/c814_01.html"}], "ResponsibleOffice": {"Name": [{"Language": "de", "Text": "Bundesamt f\u00fcr Verkehr BAV"}], "OfficeAtWeb": [{"Language": "de", "Text": "https://www.bav.admin.ch/bav/de/home.html"}]}, "Abbreviation": [{"Language": "de", "Text": "USG"}], "OfficialNumber": [{"Language": "de", "Text": "SR 814.01"}]}, {"Type": {"Code": "Law", "Text": [{"Language": "de", "Text": "Gesetzliche Grundlage"}]}, "Index": 1, "Title": [{"Language": "de", "Text": "Bundesgesetz \u00fcber die Raumplanung (Raumplanungsgesetz, RPG)"}], "Lawstatus": {"Code": "inForce", "Text": [{"Language": "de", "Text": "Rechtskr\u00e4ftig"}]}, "TextAtWeb": [{"Language": "de", "Text": "http://www.lexfind.ch/dtah/167348/2"}], "ResponsibleOffice": {"Name": [{"Language": "de", "Text": "Bundesamt f\u00fcr Verkehr BAV"}], "OfficeAtWeb": [{"Language": "de", "Text": "https://www.bav.admin.ch/bav/de/home.html"}]}, "Abbreviation": [{"Language": "de", "Text": "AltlV"}], "OfficialNumber": [{"Language": "de", "Text": "SR 700"}]}, {"Type": {"Code": "Law", "Text": [{"Language": "de", "Text": "Gesetzliche Grundlage"}]}, "Index": 1, "Title": [{"Language": "de", "Text": "Umweltschutzgesetz"}], "Lawstatus": {"Code": "inForce", "Text": [{"Language": "de", "Text": "Rechtskr\u00e4ftig"}]}, "TextAtWeb": [{"Language": "de", "Text": "http://www.admin.ch/ch/d/sr/c814_01.html"}], "ResponsibleOffice": {"Name": [{"Language": "de", "Text": "Bundesamt f\u00fcr Verkehr BAV"}], "OfficeAtWeb": [{"Language": "de", "Text": "https://www.bav.admin.ch/bav/de/home.html"}]}, "Abbreviation": [{"Language": "de", "Text": "USG"}], "OfficialNumber": [{"Language": "de", "Text": "SR 814.01"}]}, {"Type": {"Code": "Hint", "Text": [{"Language": "de", "Text": "Hinweis"}]}, "Index": 0, "Title": [{"Language": "de", "Text": "Umweltschutzgesetz"}], "Lawstatus": {"Code": "inForce", "Text": [{"Language": "de", "Text": "Rechtskr\u00e4ftig"}]}, "TextAtWeb": [{"Language": "de", "Text": "http://www.admin.ch/ch/d/sr/c814_01.html"}], "ResponsibleOffice": {"Name": [{"Language": "de", "Text": "Bundesamt f\u00fcr Verkehr BAV"}], "OfficeAtWeb": [{"Language": "de", "Text": "https://www.bav.admin.ch/bav/de/home.html"}]}, "Abbreviation": [{"Language": "de", "Text": "USG"}], "OfficialNumber": [{"Language": "de", "Text": "SR 814.01"}]}]}, {"LegendText": [{"Language": "de", "Text": "Belastet, untersuchungsbed\u00fcrftig"}], "Theme": {"Code": "ch.BelasteteStandorteOeffentlicherVerkehr", "Text": [{"Language": "de", "Text": "Kataster der belasteten Standorte im Bereich des \u00f6ffentlichen Verkehrs"}]}, "Lawstatus": {"Code": "inForce", "Text": [{"Language": "de", "Text": "Rechtskr\u00e4ftig"}]}, "ResponsibleOffice": {"Name": [{"Language": "de", "Text": "Bundesamt f\u00fcr Verkehr BAV"}], "OfficeAtWeb": [{"Language": "de", "Text": "https://www.bav.admin.ch/bav/de/home.html"}]}, "Map": {"ReferenceWMS": [{"Language": "de", "Text": "https://wms.geo.admin.ch/?SERVICE=WMS&REQUEST=GetMap&VERSION=1.3.0&STYLES=default&CRS=EPSG%3A2056&BBOX=2608564.1173571427%2C1261670.8114999998%2C2609194.3826428573%2C1262028.7715&WIDTH=493&HEIGHT=280&FORMAT=image%2Fpng&LAYERS=ch.bav.kataster-belasteter-standorte-oev.oereb&SRS=EPSG%3A2056"}], "layerIndex": 1, "layerOpacity": 0.75, "min": {"type": "Point", "coordinates": [2608564.1173571427, 1261670.8114999998], "crs": "EPSG:2056"}, "max": {"type": "Point", "coordinates": [2609194.3826428573, 1262028.7715], "crs": "EPSG:2056"}}, "SymbolRef": "http://localhost:6543/oereb/image/symbol/ch.BelasteteStandorteOeffentlicherVerkehr/1/StaoTyp2.png", "AreaShare": 3608, "TypeCode": "StaoTyp2", "TypeCodelist": "https://models.geo.admin.ch/BAFU/KbS_Codetexte_V1_4.xml", "PartInPercent": 10.3, "LegalProvisions": [{"Type": {"Code": "Law", "Text": [{"Language": "de", "Text": "Gesetzliche Grundlage"}]}, "Index": 410, "Title": [{"Language": "de", "Text": "Bundesgesetz \u00fcber den Umweltschutz"}], "Lawstatus": {"Code": "inForce", "Text": [{"Language": "de", "Text": "Rechtskr\u00e4ftig"}]}, "TextAtWeb": [{"Language": "de", "Text": "https://www.admin.ch/ch/d/sr/c814_01.html"}], "ResponsibleOffice": {"Name": [{"Language": "de", "Text": "Bundeskanzlei"}], "OfficeAtWeb": [{"Language": "de", "Text": "https://www.bk.admin.ch"}], "UID": "CHE263297189"}, "Abbreviation": [{"Language": "de", "Text": "USG"}], "OfficialNumber": [{"Language": "de", "Text": "SR 814.01"}]}, {"Type": {"Code": "Law", "Text": [{"Language": "de", "Text": "Gesetzliche Grundlage"}]}, "Index": 420, "Title": [{"Language": "de", "Text": "Verordnung \u00fcber die Sanierung von belasteten Standorten"}], "Lawstatus": {"Code": "inForce", "Text": [{"Language": "de", "Text": "Rechtskr\u00e4ftig"}]}, "TextAtWeb": [{"Language": "de", "Text": "https://www.admin.ch/ch/d/sr/c814_680.html"}], "ResponsibleOffice": {"Name": [{"Language": "de", "Text": "Bundeskanzlei"}], "OfficeAtWeb": [{"Language": "de", "Text": "https://www.bk.admin.ch"}], "UID": "CHE263297189"}, "Abbreviation": [{"Language": "de", "Text": "AltlV"}], "OfficialNumber": [{"Language": "de", "Text": "SR 814.680"}]}, {"Type": {"Code": "LegalProvision", "Text": [{"Language": "de", "Text": "Rechtsvorschrift"}]}, "Index": 2, "Title": [{"Language": "de", "Text": "B61001"}], "Lawstatus": {"Code": "inForce", "Text": [{"Language": "de", "Text": "Rechtskr\u00e4ftig"}]}, "TextAtWeb": [{"Language": "de", "Text": "https://www.bav.admin.ch/dam/bav/de/dokumente/das-bav/finanzierung/faktenblatt_fernverkehr_konzessionierung.pdf.download.pdf/d_Faktenblatt_Fernverkehr_und_Konzessionierung%20.pdf"}], "ResponsibleOffice": {"Name": [{"Language": "de", "Text": "Bundesamt f\u00fcr Verkehr BAV"}], "OfficeAtWeb": [{"Language": "de", "Text": "https://www.bav.admin.ch/bav/de/home.html"}]}}, {"Type": {"Code": "Law", "Text": [{"Language": "de", "Text": "Gesetzliche Grundlage"}]}, "Index": 1, "Title": [{"Language": "de", "Text": "Umweltschutzgesetz"}], "Lawstatus": {"Code": "inForce", "Text": [{"Language": "de", "Text": "Rechtskr\u00e4ftig"}]}, "TextAtWeb": [{"Language": "de", "Text": "http://www.admin.ch/ch/d/sr/c814_01.html"}], "ResponsibleOffice": {"Name": [{"Language": "de", "Text": "Bundesamt f\u00fcr Verkehr BAV"}], "OfficeAtWeb": [{"Language": "de", "Text": "https://www.bav.admin.ch/bav/de/home.html"}]}, "Abbreviation": [{"Language": "de", "Text": "USG"}], "OfficialNumber": [{"Language": "de", "Text": "SR 814.01"}]}, {"Type": {"Code": "Law", "Text": [{"Language": "de", "Text": "Gesetzliche Grundlage"}]}, "Index": 1, "Title": [{"Language": "de", "Text": "Altlasten-Verordnung"}], "Lawstatus": {"Code": "inForce", "Text": [{"Language": "de", "Text": "Rechtskr\u00e4ftig"}]}, "TextAtWeb": [{"Language": "de", "Text": "http://www.admin.ch/ch/d/sr/c814_680.html"}], "ResponsibleOffice": {"Name": [{"Language": "de", "Text": "Bundesamt f\u00fcr Verkehr BAV"}], "OfficeAtWeb": [{"Language": "de", "Text": "https://www.bav.admin.ch/bav/de/home.html"}]}, "Abbreviation": [{"Language": "de", "Text": "AltlV"}], "OfficialNumber": [{"Language": "de", "Text": "SR 814.680"}]}]}, {"LegendText": [{"Language": "de", "Text": "Belastet, untersuchungsbed\u00fcrftig"}], "Theme": {"Code": "ch.BelasteteStandorteOeffentlicherVerkehr", "Text": [{"Language": "de", "Text": "Kataster der belasteten Standorte im Bereich des \u00f6ffentlichen Verkehrs"}]}, "Lawstatus": {"Code": "changeWithPreEffect", "Text": [{"Language": "de", "Text": "\u00c4nderung mit Vorwirkung"}]}, "ResponsibleOffice": {"Name": [{"Language": "de", "Text": "Bundesamt f\u00fcr Verkehr BAV"}], "OfficeAtWeb": [{"Language": "de", "Text": "https://www.bav.admin.ch/bav/de/home.html"}]}, "Map": {"ReferenceWMS": [{"Language": "de", "Text": "https://wms.geo.admin.ch/?SERVICE=WMS&REQUEST=GetMap&VERSION=1.3.0&STYLES=default&CRS=EPSG%3A2056&BBOX=2608564.1173571427%2C1261670.8114999998%2C2609194.3826428573%2C1262028.7715&WIDTH=493&HEIGHT=280&FORMAT=image%2Fpng&LAYERS=ch.bav.kataster-belasteter-standorte-oev.oereb&SRS=EPSG%3A2056"}], "layerIndex": 1, "layerOpacity": 0.75, "min": {"type": "Point", "coordinates": [2608564.1173571427, 1261670.8114999998], "crs": "EPSG:2056"}, "max": {"type": "Point", "coordinates": [2609194.3826428573, 1262028.7715], "crs": "EPSG:2056"}}, "SymbolRef": "http://localhost:6543/oereb/image/symbol/ch.BelasteteStandorteOeffentlicherVerkehr/1/StaoTyp3.png", "LengthShare": 101, "TypeCode": "StaoTyp3", "TypeCodelist": "https://models.geo.admin.ch/BAFU/KbS_Codetexte_V1_4.xml", "LegalProvisions": [{"Type": {"Code": "Law", "Text": [{"Language": "de", "Text": "Gesetzliche Grundlage"}]}, "Index": 410, "Title": [{"Language": "de", "Text": "Bundesgesetz \u00fcber den Umweltschutz"}], "Lawstatus": {"Code": "inForce", "Text": [{"Language": "de", "Text": "Rechtskr\u00e4ftig"}]}, "TextAtWeb": [{"Language": "de", "Text": "https://www.admin.ch/ch/d/sr/c814_01.html"}], "ResponsibleOffice": {"Name": [{"Language": "de", "Text": "Bundeskanzlei"}], "OfficeAtWeb": [{"Language": "de", "Text": "https://www.bk.admin.ch"}], "UID": "CHE263297189"}, "Abbreviation": [{"Language": "de", "Text": "USG"}], "OfficialNumber": [{"Language": "de", "Text": "SR 814.01"}]}, {"Type": {"Code": "Law", "Text": [{"Language": "de", "Text": "Gesetzliche Grundlage"}]}, "Index": 420, "Title": [{"Language": "de", "Text": "Verordnung \u00fcber die Sanierung von belasteten Standorten"}], "Lawstatus": {"Code": "inForce", "Text": [{"Language": "de", "Text": "Rechtskr\u00e4ftig"}]}, "TextAtWeb": [{"Language": "de", "Text": "https://www.admin.ch/ch/d/sr/c814_680.html"}], "ResponsibleOffice": {"Name": [{"Language": "de", "Text": "Bundeskanzlei"}], "OfficeAtWeb": [{"Language": "de", "Text": "https://www.bk.admin.ch"}], "UID": "CHE263297189"}, "Abbreviation": [{"Language": "de", "Text": "AltlV"}], "OfficialNumber": [{"Language": "de", "Text": "SR 814.680"}]}, {"Type": {"Code": "LegalProvision", "Text": [{"Language": "de", "Text": "Rechtsvorschrift"}]}, "Index": 2, "Title": [{"Language": "de", "Text": "B61001"}], "Lawstatus": {"Code": "inForce", "Text": [{"Language": "de", "Text": "Rechtskr\u00e4ftig"}]}, "TextAtWeb": [{"Language": "de", "Text": "https://www.bav.admin.ch/dam/bav/de/dokumente/das-bav/finanzierung/faktenblatt_fernverkehr_konzessionierung.pdf.download.pdf/d_Faktenblatt_Fernverkehr_und_Konzessionierung%20.pdf"}], "ResponsibleOffice": {"Name": [{"Language": "de", "Text": "Bundesamt f\u00fcr Verkehr BAV"}], "OfficeAtWeb": [{"Language": "de", "Text": "https://www.bav.admin.ch/bav/de/home.html"}]}}, {"Type": {"Code": "Law", "Text": [{"Language": "de", "Text": "Gesetzliche Grundlage"}]}, "Index": 1, "Title": [{"Language": "de", "Text": "Umweltschutzgesetz"}], "Lawstatus": {"Code": "inForce", "Text": [{"Language": "de", "Text": "Rechtskr\u00e4ftig"}]}, "TextAtWeb": [{"Language": "de", "Text": "http://www.admin.ch/ch/d/sr/c814_01.html"}], "ResponsibleOffice": {"Name": [{"Language": "de", "Text": "Bundesamt f\u00fcr Verkehr BAV"}], "OfficeAtWeb": [{"Language": "de", "Text": "https://www.bav.admin.ch/bav/de/home.html"}]}, "Abbreviation": [{"Language": "de", "Text": "USG"}], "OfficialNumber": [{"Language": "de", "Text": "SR 814.01"}]}, {"Type": {"Code": "Law", "Text": [{"Language": "de", "Text": "Gesetzliche Grundlage"}]}, "Index": 1, "Title": [{"Language": "de", "Text": "Altlasten-Verordnung"}], "Lawstatus": {"Code": "inForce", "Text": [{"Language": "de", "Text": "Rechtskr\u00e4ftig"}]}, "TextAtWeb": [{"Language": "de", "Text": "http://www.admin.ch/ch/d/sr/c814_680.html"}], "ResponsibleOffice": {"Name": [{"Language": "de", "Text": "Bundesamt f\u00fcr Verkehr BAV"}], "OfficeAtWeb": [{"Language": "de", "Text": "https://www.bav.admin.ch/bav/de/home.html"}]}, "Abbreviation": [{"Language": "de", "Text": "AltlV"}], "OfficialNumber": [{"Language": "de", "Text": "SR 814.680"}]}]}, {"LegendText": [{"Language": "de", "Text": "Belastet, untersuchungsbed\u00fcrftig"}], "Theme": {"Code": "ch.BelasteteStandorteOeffentlicherVerkehr", "Text": [{"Language": "de", "Text": "Kataster der belasteten Standorte im Bereich des \u00f6ffentlichen Verkehrs"}]}, "Lawstatus": {"Code": "changeWithoutPreEffect", "Text": [{"Language": "de", "Text": "\u00c4nderung ohne Vorwirkung"}]}, "ResponsibleOffice": {"Name": [{"Language": "de", "Text": "Bundesamt f\u00fcr Verkehr BAV"}], "OfficeAtWeb": [{"Language": "de", "Text": "https://www.bav.admin.ch/bav/de/home.html"}]}, "Map": {"ReferenceWMS": [{"Language": "de", "Text": "https://wms.geo.admin.ch/?SERVICE=WMS&REQUEST=GetMap&VERSION=1.3.0&STYLES=default&CRS=EPSG%3A2056&BBOX=2608564.1173571427%2C1261670.8114999998%2C2609194.3826428573%2C1262028.7715&WIDTH=493&HEIGHT=280&FORMAT=image%2Fpng&LAYERS=ch.bav.kataster-belasteter-standorte-oev.oereb&SRS=EPSG%3A2056"}], "layerIndex": 1, "layerOpacity": 0.75, "min": {"type": "Point", "coordinates": [2608564.1173571427, 1261670.8114999998], "crs": "EPSG:2056"}, "max": {"type": "Point", "coordinates": [2609194.3826428573, 1262028.7715], "crs": "EPSG:2056"}}, "SymbolRef": "http://localhost:6543/oereb/image/symbol/ch.BelasteteStandorteOeffentlicherVerkehr/1/StaoTyp3.png", "LengthShare": 63, "TypeCode": "StaoTyp3", "TypeCodelist": "https://models.geo.admin.ch/BAFU/KbS_Codetexte_V1_4.xml", "LegalProvisions": [{"Type": {"Code": "Law", "Text": [{"Language": "de", "Text": "Gesetzliche Grundlage"}]}, "Index": 410, "Title": [{"Language": "de", "Text": "Bundesgesetz \u00fcber den Umweltschutz"}], "Lawstatus": {"Code": "inForce", "Text": [{"Language": "de", "Text": "Rechtskr\u00e4ftig"}]}, "TextAtWeb": [{"Language": "de", "Text": "https://www.admin.ch/ch/d/sr/c814_01.html"}], "ResponsibleOffice": {"Name": [{"Language": "de", "Text": "Bundeskanzlei"}], "OfficeAtWeb": [{"Language": "de", "Text": "https://www.bk.admin.ch"}], "UID": "CHE263297189"}, "Abbreviation": [{"Language": "de", "Text": "USG"}], "OfficialNumber": [{"Language": "de", "Text": "SR 814.01"}]}, {"Type": {"Code": "Law", "Text": [{"Language": "de", "Text": "Gesetzliche Grundlage"}]}, "Index": 420, "Title": [{"Language": "de", "Text": "Verordnung \u00fcber die Sanierung von belasteten Standorten"}], "Lawstatus": {"Code": "inForce", "Text": [{"Language": "de", "Text": "Rechtskr\u00e4ftig"}]}, "TextAtWeb": [{"Language": "de", "Text": "https://www.admin.ch/ch/d/sr/c814_680.html"}], "ResponsibleOffice": {"Name": [{"Language": "de", "Text": "Bundeskanzlei"}], "OfficeAtWeb": [{"Language": "de", "Text": "https://www.bk.admin.ch"}], "UID": "CHE263297189"}, "Abbreviation": [{"Language": "de", "Text": "AltlV"}], "OfficialNumber": [{"Language": "de", "Text": "SR 814.680"}]}, {"Type": {"Code": "LegalProvision", "Text": [{"Language": "de", "Text": "Rechtsvorschrift"}]}, "Index": 2, "Title": [{"Language": "de", "Text": "B61001"}], "Lawstatus": {"Code": "inForce", "Text": [{"Language": "de", "Text": "Rechtskr\u00e4ftig"}]}, "TextAtWeb": [{"Language": "de", "Text": "https://www.bav.admin.ch/dam/bav/de/dokumente/das-bav/finanzierung/faktenblatt_fernverkehr_konzessionierung.pdf.download.pdf/d_Faktenblatt_Fernverkehr_und_Konzessionierung%20.pdf"}], "ResponsibleOffice": {"Name": [{"Language": "de", "Text": "Bundesamt f\u00fcr Verkehr BAV"}], "OfficeAtWeb": [{"Language": "de", "Text": "https://www.bav.admin.ch/bav/de/home.html"}]}}, {"Type": {"Code": "Law", "Text": [{"Language": "de", "Text": "Gesetzliche Grundlage"}]}, "Index": 1, "Title": [{"Language": "de", "Text": "Umweltschutzgesetz"}], "Lawstatus": {"Code": "inForce", "Text": [{"Language": "de", "Text": "Rechtskr\u00e4ftig"}]}, "TextAtWeb": [{"Language": "de", "Text": "http://www.admin.ch/ch/d/sr/c814_01.html"}], "ResponsibleOffice": {"Name": [{"Language": "de", "Text": "Bundesamt f\u00fcr Verkehr BAV"}], "OfficeAtWeb": [{"Language": "de", "Text": "https://www.bav.admin.ch/bav/de/home.html"}]}, "Abbreviation": [{"Language": "de", "Text": "USG"}], "OfficialNumber": [{"Language": "de", "Text": "SR 814.01"}]}, {"Type": {"Code": "Law", "Text": [{"Language": "de", "Text": "Gesetzliche Grundlage"}]}, "Index": 1, "Title": [{"Language": "de", "Text": "Altlasten-Verordnung"}], "Lawstatus": {"Code": "inForce", "Text": [{"Language": "de", "Text": "Rechtskr\u00e4ftig"}]}, "TextAtWeb": [{"Language": "de", "Text": "http://www.admin.ch/ch/d/sr/c814_680.html"}], "ResponsibleOffice": {"Name": [{"Language": "de", "Text": "Bundesamt f\u00fcr Verkehr BAV"}], "OfficeAtWeb": [{"Language": "de", "Text": "https://www.bav.admin.ch/bav/de/home.html"}]}, "Abbreviation": [{"Language": "de", "Text": "AltlV"}], "OfficialNumber": [{"Language": "de", "Text": "SR 814.680"}]}]}, {"LegendText": [{"Language": "de", "Text": "Belastet, untersuchungsbed\u00fcrftig"}], "Theme": {"Code": "ch.BelasteteStandorteOeffentlicherVerkehr", "Text": [{"Language": "de", "Text": "Kataster der belasteten Standorte im Bereich des \u00f6ffentlichen Verkehrs"}]}, "Lawstatus": {"Code": "inForce", "Text": [{"Language": "de", "Text": "Rechtskr\u00e4ftig"}]}, "ResponsibleOffice": {"Name": [{"Language": "de", "Text": "Bundesamt f\u00fcr Verkehr BAV"}], "OfficeAtWeb": [{"Language": "de", "Text": "https://www.bav.admin.ch/bav/de/home.html"}]}, "Map": {"ReferenceWMS": [{"Language": "de", "Text": "https://wms.geo.admin.ch/?SERVICE=WMS&REQUEST=GetMap&VERSION=1.3.0&STYLES=default&CRS=EPSG%3A2056&BBOX=2608564.1173571427%2C1261670.8114999998%2C2609194.3826428573%2C1262028.7715&WIDTH=493&HEIGHT=280&FORMAT=image%2Fpng&LAYERS=ch.bav.kataster-belasteter-standorte-oev.oereb&SRS=EPSG%3A2056"}], "layerIndex": 1, "layerOpacity": 0.75, "min": {"type": "Point", "coordinates": [2608564.1173571427, 1261670.8114999998], "crs": "EPSG:2056"}, "max": {"type": "Point", "coordinates": [2609194.3826428573, 1262028.7715], "crs": "EPSG:2056"}}, "SymbolRef": "http://localhost:6543/oereb/image/symbol/ch.BelasteteStandorteOeffentlicherVerkehr/1/StaoTyp1.png", "NrOfPoints": 2, "TypeCode": "StaoTyp1", "TypeCodelist": "https://models.geo.admin.ch/BAFU/KbS_Codetexte_V1_4.xml", "LegalProvisions": [{"Type": {"Code": "Law", "Text": [{"Language": "de", "Text": "Gesetzliche Grundlage"}]}, "Index": 410, "Title": [{"Language": "de", "Text": "Bundesgesetz \u00fcber den Umweltschutz"}], "Lawstatus": {"Code": "inForce", "Text": [{"Language": "de", "Text": "Rechtskr\u00e4ftig"}]}, "TextAtWeb": [{"Language": "de", "Text": "https://www.admin.ch/ch/d/sr/c814_01.html"}], "ResponsibleOffice": {"Name": [{"Language": "de", "Text": "Bundeskanzlei"}], "OfficeAtWeb": [{"Language": "de", "Text": "https://www.bk.admin.ch"}], "UID": "CHE263297189"}, "Abbreviation": [{"Language": "de", "Text": "USG"}], "OfficialNumber": [{"Language": "de", "Text": "SR 814.01"}]}, {"Type": {"Code": "Law", "Text": [{"Language": "de", "Text": "Gesetzliche Grundlage"}]}, "Index": 420, "Title": [{"Language": "de", "Text": "Verordnung \u00fcber die Sanierung von belasteten Standorten"}], "Lawstatus": {"Code": "inForce", "Text": [{"Language": "de", "Text": "Rechtskr\u00e4ftig"}]}, "TextAtWeb": [{"Language": "de", "Text": "https://www.admin.ch/ch/d/sr/c814_680.html"}], "ResponsibleOffice": {"Name": [{"Language": "de", "Text": "Bundeskanzlei"}], "OfficeAtWeb": [{"Language": "de", "Text": "https://www.bk.admin.ch"}], "UID": "CHE263297189"}, "Abbreviation": [{"Language": "de", "Text": "AltlV"}], "OfficialNumber": [{"Language": "de", "Text": "SR 814.680"}]}, {"Type": {"Code": "LegalProvision", "Text": [{"Language": "de", "Text": "Rechtsvorschrift"}]}, "Index": 2, "Title": [{"Language": "de", "Text": "B61001"}], "Lawstatus": {"Code": "inForce", "Text": [{"Language": "de", "Text": "Rechtskr\u00e4ftig"}]}, "TextAtWeb": [{"Language": "de", "Text": "https://www.bav.admin.ch/dam/bav/de/dokumente/das-bav/finanzierung/faktenblatt_fernverkehr_konzessionierung.pdf.download.pdf/d_Faktenblatt_Fernverkehr_und_Konzessionierung%20.pdf"}], "ResponsibleOffice": {"Name": [{"Language": "de", "Text": "Bundesamt f\u00fcr Verkehr BAV"}], "OfficeAtWeb": [{"Language": "de", "Text": "https://www.bav.admin.ch/bav/de/home.html"}]}}, {"Type": {"Code": "Law", "Text": [{"Language": "de", "Text": "Gesetzliche Grundlage"}]}, "Index": 1, "Title": [{"Language": "de", "Text": "Umweltschutzgesetz"}], "Lawstatus": {"Code": "inForce", "Text": [{"Language": "de", "Text": "Rechtskr\u00e4ftig"}]}, "TextAtWeb": [{"Language": "de", "Text": "http://www.admin.ch/ch/d/sr/c814_01.html"}], "ResponsibleOffice": {"Name": [{"Language": "de", "Text": "Bundesamt f\u00fcr Verkehr BAV"}], "OfficeAtWeb": [{"Language": "de", "Text": "https://www.bav.admin.ch/bav/de/home.html"}]}, "Abbreviation": [{"Language": "de", "Text": "USG"}], "OfficialNumber": [{"Language": "de", "Text": "SR 814.01"}]}, {"Type": {"Code": "Law", "Text": [{"Language": "de", "Text": "Gesetzliche Grundlage"}]}, "Index": 1, "Title": [{"Language": "de", "Text": "Altlasten-Verordnung"}], "Lawstatus": {"Code": "inForce", "Text": [{"Language": "de", "Text": "Rechtskr\u00e4ftig"}]}, "TextAtWeb": [{"Language": "de", "Text": "http://www.admin.ch/ch/d/sr/c814_680.html"}], "ResponsibleOffice": {"Name": [{"Language": "de", "Text": "Bundesamt f\u00fcr Verkehr BAV"}], "OfficeAtWeb": [{"Language": "de", "Text": "https://www.bav.admin.ch/bav/de/home.html"}]}, "Abbreviation": [{"Language": "de", "Text": "AltlV"}], "OfficialNumber": [{"Language": "de", "Text": "SR 814.680"}]}]}, {"LegendText": [{"Language": "de", "Text": "Fassungsbereich (Zone S1)"}], "Theme": {"Code": "ch.Grundwasserschutzzonen", "Text": [{"Language": "de", "Text": "Grundwasserschutzzonen"}]}, "Lawstatus": {"Code": "inForce", "Text": [{"Language": "de", "Text": "Rechtskr\u00e4ftig"}]}, "ResponsibleOffice": {"Name": [{"Language": "de", "Text": "Bundesamt f\u00fcr Verkehr BAV"}], "OfficeAtWeb": [{"Language": "de", "Text": "https://www.bav.admin.ch/bav/de/home.html"}]}, "Map": {"ReferenceWMS": [{"Language": "de", "Text": "https://wms.geo.admin.ch/?SERVICE=WMS&REQUEST=GetMap&VERSION=1.3.0&STYLES=default&CRS=EPSG%3A2056&BBOX=2608564.1173571427%2C1261670.8114999998%2C2609194.3826428573%2C1262028.7715&WIDTH=493&HEIGHT=280&FORMAT=image%2Fpng&LAYERS=ch.bav.kataster-belasteter-standorte-oev.oereb&SRS=EPSG%3A2056"}], "layerIndex": 1, "layerOpacity": 0.75, "min": {"type": "Point", "coordinates": [2608564.1173571427, 1261670.8114999998], "crs": "EPSG:2056"}, "max": {"type": "Point", "coordinates": [2609194.3826428573, 1262028.7715], "crs": "EPSG:2056"}}, "SymbolRef": "http://localhost:6543/oereb/image/symbol/ch.Grundwasserschutzzonen/1/StaoTyp2.png", "AreaShare": 7824, "TypeCode": "StaoTyp2", "TypeCodelist": "https://models.geo.admin.ch/BAFU/KbS_Codetexte_V1_4.xml", "PartInPercent": 22.3, "LegalProvisions": [{"Type": {"Code": "Law", "Text": [{"Language": "de", "Text": "Gesetzliche Grundlage"}]}, "Index": 510, "Title": [{"Language": "de", "Text": "Bundesgesetz \u00fcber den Schutz der Gew\u00e4sser"}], "Lawstatus": {"Code": "inForce", "Text": [{"Language": "de", "Text": "Rechtskr\u00e4ftig"}]}, "TextAtWeb": [{"Language": "de", "Text": "https://www.admin.ch/ch/d/sr/c814_20.html"}], "ResponsibleOffice": {"Name": [{"Language": "de", "Text": "Bundeskanzlei"}], "OfficeAtWeb": [{"Language": "de", "Text": "https://www.bk.admin.ch"}], "UID": "CHE263297189"}, "Abbreviation": [{"Language": "de", "Text": "GSchG"}], "OfficialNumber": [{"Language": "de", "Text": "SR 814.20"}]}, {"Type": {"Code": "Law", "Text": [{"Language": "de", "Text": "Gesetzliche Grundlage"}]}, "Index": 520, "Title": [{"Language": "de", "Text": "Gew\u00e4sserschutzverordnung"}], "Lawstatus": {"Code": "inForce", "Text": [{"Language": "de", "Text": "Rechtskr\u00e4ftig"}]}, "TextAtWeb": [{"Language": "de", "Text": "https://www.admin.ch/ch/d/sr/c814_201.html"}], "ResponsibleOffice": {"Name": [{"Language": "de", "Text": "Bundeskanzlei"}], "OfficeAtWeb": [{"Language": "de", "Text": "https://www.bk.admin.ch"}], "UID": "CHE263297189"}, "Abbreviation": [{"Language": "de", "Text": "GSchV"}], "OfficialNumber": [{"Language": "de", "Text": "SR 814.201"}]}, {"Type": {"Code": "LegalProvision", "Text": [{"Language": "de", "Text": "Rechtsvorschrift"}]}, "Index": 2, "Title": [{"Language": "de", "Text": "B61001"}], "Lawstatus": {"Code": "inForce", "Text": [{"Language": "de", "Text": "Rechtskr\u00e4ftig"}]}, "TextAtWeb": [{"Language": "de", "Text": "https://www.bav.admin.ch/dam/bav/de/dokumente/das-bav/finanzierung/faktenblatt_fernverkehr_konzessionierung.pdf.download.pdf/d_Faktenblatt_Fernverkehr_und_Konzessionierung%20.pdf"}], "ResponsibleOffice": {"Name": [{"Language": "de", "Text": "Bundesamt f\u00fcr Verkehr BAV"}], "OfficeAtWeb": [{"Language": "de", "Text": "https://www.bav.admin.ch/bav/de/home.html"}]}}, {"Type": {"Code": "Law", "Text": [{"Language": "de", "Text": "Gesetzliche Grundlage"}]}, "Index": 1, "Title": [{"Language": "de", "Text": "Umweltschutzgesetz"}], "Lawstatus": {"Code": "inForce", "Text": [{"Language": "de", "Text": "Rechtskr\u00e4ftig"}]}, "TextAtWeb": [{"Language": "de", "Text": "http://www.admin.ch/ch/d/sr/c814_01.html"}], "ResponsibleOffice": {"Name": [{"Language": "de", "Text": "Bundesamt f\u00fcr Verkehr BAV"}], "OfficeAtWeb": [{"Language": "de", "Text": "https://www.bav.admin.ch/bav/de/home.html"}]}, "Abbreviation": [{"Language": "de", "Text": "USG"}], "OfficialNumber": [{"Language": "de", "Text": "SR 814.01"}]}, {"Type": {"Code": "Law", "Text": [{"Language": "de", "Text": "Gesetzliche Grundlage"}]}, "Index": 1, "Title": [{"Language": "de", "Text": "Altlasten-Verordnung"}], "Lawstatus": {"Code": "inForce", "Text": [{"Language": "de", "Text": "Rechtskr\u00e4ftig"}]}, "TextAtWeb": [{"Language": "de", "Text": "http://www.admin.ch/ch/d/sr/c814_680.html"}], "ResponsibleOffice": {"Name": [{"Language": "de", "Text": "Bundesamt f\u00fcr Verkehr BAV"}], "OfficeAtWeb": [{"Language": "de", "Text": "https://www.bav.admin.ch/bav/de/home.html"}]}, "Abbreviation": [{"Language": "de", "Text": "AltlV"}], "OfficialNumber": [{"Language": "de", "Text": "SR 814.680"}]}]}, {"LegendText": [{"Language": "de", "Text": "Belastet, untersuchungsbed\u00fcrftig"}], "Theme": {"Code": "ch.StatischeWaldgrenzen", "Text": [{"Language": "de", "Text": "Statische Waldgrenzen"}]}, "Lawstatus": {"Code": "inForce", "Text": [{"Language": "de", "Text": "Rechtskr\u00e4ftig"}]}, "ResponsibleOffice": {"Name": [{"Language": "de", "Text": "Bundesamt f\u00fcr Verkehr BAV"}], "OfficeAtWeb": [{"Language": "de", "Text": "https://www.bav.admin.ch/bav/de/home.html"}]}, "Map": {"ReferenceWMS": [{"Language": "de", "Text": "https://wms.geo.admin.ch/?SERVICE=WMS&REQUEST=GetMap&VERSION=1.3.0&STYLES=default&CRS=EPSG%3A2056&BBOX=2608564.1173571427%2C1261670.8114999998%2C2609194.3826428573%2C1262028.7715&WIDTH=493&HEIGHT=280&FORMAT=image%2Fpng&LAYERS=ch.bav.kataster-belasteter-standorte-oev.oereb&SRS=EPSG%3A2056"}], "layerIndex": 1, "layerOpacity": 0.75, "min": {"type": "Point", "coordinates": [2608564.1173571427, 1261670.8114999998], "crs": "EPSG:2056"}, "max": {"type": "Point", "coordinates": [2609194.3826428573, 1262028.7715], "crs": "EPSG:2056"}}, "SymbolRef": "http://localhost:6543/oereb/image/symbol/ch.StatischeWaldgrenzen/1/StaoTyp2.png", "LengthShare": 101, "TypeCode": "StaoTyp2", "TypeCodelist": "https://models.geo.admin.ch/BAFU/KbS_Codetexte_V1_4.xml", "LegalProvisions": [{"Type": {"Code": "Law", "Text": [{"Language": "de", "Text": "Gesetzliche Grundlage"}]}, "Index": 710, "Title": [{"Language": "de", "Text": "Bundesgesetz \u00fcber den Wald"}], "Lawstatus": {"Code": "inForce", "Text": [{"Language": "de", "Text": "Rechtskr\u00e4ftig"}]}, "TextAtWeb": [{"Language": "de", "Text": "https://www.admin.ch/ch/d/sr/c921_0.html"}], "ResponsibleOffice": {"Name": [{"Language": "de", "Text": "Bundeskanzlei"}], "OfficeAtWeb": [{"Language": "de", "Text": "https://www.bk.admin.ch"}], "UID": "CHE263297189"}, "Abbreviation": [{"Language": "de", "Text": "WaG"}], "OfficialNumber": [{"Language": "de", "Text": "SR 921.0"}]}, {"Type": {"Code": "Law", "Text": [{"Language": "de", "Text": "Gesetzliche Grundlage"}]}, "Index": 720, "Title": [{"Language": "de", "Text": "Verordnung \u00fcber den Wald"}], "Lawstatus": {"Code": "inForce", "Text": [{"Language": "de", "Text": "Rechtskr\u00e4ftig"}]}, "TextAtWeb": [{"Language": "de", "Text": "https://www.admin.ch/ch/d/sr/c921_01.html"}], "ResponsibleOffice": {"Name": [{"Language": "de", "Text": "Bundeskanzlei"}], "OfficeAtWeb": [{"Language": "de", "Text": "https://www.bk.admin.ch"}], "UID": "CHE263297189"}, "Abbreviation": [{"Language": "de", "Text": "WaV"}], "OfficialNumber": [{"Language": "de", "Text": "SR 921.01"}]}, {"Type": {"Code": "LegalProvision", "Text": [{"Language": "de", "Text": "Rechtsvorschrift"}]}, "Index": 2, "Title": [{"Language": "de", "Text": "Baugesetz, 3891.100"}], "Lawstatus": {"Code": "inForce", "Text": [{"Language": "de", "Text": "Rechtskr\u00e4ftig"}]}, "TextAtWeb": [{"Language": "de", "Text": "https://oereb-gr-preview.000.ch/api/attachments/197"}], "ResponsibleOffice": {"Name": [{"Language": "de", "Text": "Bundesamt f\u00fcr Verkehr BAV"}], "OfficeAtWeb": [{"Language": "de", "Text": "https://www.bav.admin.ch/bav/de/home.html"}]}}, {"Type": {"Code": "Law", "Text": [{"Language": "de", "Text": "Gesetzliche Grundlage"}]}, "Index": 1, "Title": [{"Language": "de", "Text": "Raumplanungsverordnung f\u00fcr den Kanton Graub\u00fcnden"}], "Lawstatus": {"Code": "inForce", "Text": [{"Language": "de", "Text": "Rechtskr\u00e4ftig"}]}, "TextAtWeb": [{"Language": "de", "Text": "https://www.gr-lex.gr.ch/frontend/versions/pdf_file_with_annex/2934?locale=de"}], "ResponsibleOffice": {"Name": [{"Language": "de", "Text": "Bundesamt f\u00fcr Verkehr BAV"}], "OfficeAtWeb": [{"Language": "de", "Text": "https://www.bav.admin.ch/bav/de/home.html"}]}, "Abbreviation": [{"Language": "de", "Text": "KRVO"}], "OfficialNumber": [{"Language": "de", "Text": "BR 801.110"}]}, {"Type": {"Code": "Law", "Text": [{"Language": "de", "Text": "Gesetzliche Grundlage"}]}, "Index": 1, "Title": [{"Language": "de", "Text": "Bundesgesetz \u00fcber die Raumplanung (Raumplanungsgesetz, RPG)"}], "Lawstatus": {"Code": "inForce", "Text": [{"Language": "de", "Text": "Rechtskr\u00e4ftig"}]}, "TextAtWeb": [{"Language": "de", "Text": "http://www.lexfind.ch/dtah/167348/2"}], "ResponsibleOffice": {"Name": [{"Language": "de", "Text": "Bundesamt f\u00fcr Verkehr BAV"}], "OfficeAtWeb": [{"Language": "de", "Text": "https://www.bav.admin.ch/bav/de/home.html"}]}, "Abbreviation": [{"Language": "de", "Text": "RPG"}], "OfficialNumber": [{"Language": "de", "Text": "SR 700"}]}, {"Type": {"Code": "Law", "Text": [{"Language": "de", "Text": "Gesetzliche Grundlage"}]}, "Index": 1, "Title": [{"Language": "de", "Text": "Raumplanungsgesetz f\u00fcr den Kanton Graub\u00fcnden"}], "Lawstatus": {"Code": "inForce", "Text": [{"Language": "de", "Text": "Rechtskr\u00e4ftig"}]}, "TextAtWeb": [{"Language": "de", "Text": "https://www.gr-lex.gr.ch/frontend/versions/pdf_file_with_annex/2936?locale=de"}], "ResponsibleOffice": {"Name": [{"Language": "de", "Text": "Bundesamt f\u00fcr Verkehr BAV"}], "OfficeAtWeb": [{"Language": "de", "Text": "https://www.bav.admin.ch/bav/de/home.html"}]}, "Abbreviation": [{"Language": "de", "Text": "KRG"}], "OfficialNumber": [{"Language": "de", "Text": "BR 801.100"}]}, {"Type": {"Code": "LegalProvision", "Text": [{"Language": "de", "Text": "Rechtsvorschrift"}]}, "Index": 2, "Title": [{"Language": "de", "Text": "Revision Ortsplanung, 07.447"}], "Lawstatus": {"Code": "inForce", "Text": [{"Language": "de", "Text": "Rechtskr\u00e4ftig"}]}, "TextAtWeb": [{"Language": "de", "Text": "https://oereb-gr-preview.000.ch/api/attachments/213"}], "ResponsibleOffice": {"Name": [{"Language": "de", "Text": "Bundesamt f\u00fcr Verkehr BAV"}], "OfficeAtWeb": [{"Language": "de", "Text": "https://www.bav.admin.ch/bav/de/home.html"}]}}, {"Type": {"Code": "LegalProvision", "Text": [{"Language": "de", "Text": "Rechtsvorschrift"}]}, "Index": 2, "Title": [{"Language": "de", "Text": "Baugesetz, 3891.100"}], "Lawstatus": {"Code": "inForce", "Text": [{"Language": "de", "Text": "Rechtskr\u00e4ftig"}]}, "TextAtWeb": [{"Language": "de", "Text": "https://oereb-gr-preview.000.ch/api/attachments/198"}], "ResponsibleOffice": {"Name": [{"Language": "de", "Text": "Bundesamt f\u00fcr Verkehr BAV"}], "OfficeAtWeb": [{"Language": "de", "Text": "https://www.bav.admin.ch/bav/de/home.html"}]}}, {"Type": {"Code": "LegalProvision", "Text": [{"Language": "de", "Text": "Rechtsvorschrift"}]}, "Index": 2, "Title": [{"Language": "de", "Text": "Revision Ortsplanung, 07.447"}], "Lawstatus": {"Code": "inForce", "Text": [{"Language": "de", "Text": "Rechtskr\u00e4ftig"}]}, "TextAtWeb": [{"Language": "de", "Text": "https://oereb-gr-preview.000.ch/api/attachments/214"}], "ResponsibleOffice": {"Name": [{"Language": "de", "Text": "Bundesamt f\u00fcr Verkehr BAV"}], "OfficeAtWeb": [{"Language": "de", "Text": "https://www.bav.admin.ch/bav/de/home.html"}]}}, {"Type": {"Code": "Hint", "Text": [{"Language": "de", "Text": "Hinweis"}]}, "Index": 3, "Title": [{"Language": "de", "Text": "Guter Ratschlag"}], "Lawstatus": {"Code": "inForce", "Text": [{"Language": "de", "Text": "Rechtskr\u00e4ftig"}]}, "TextAtWeb": [{"Language": "de", "Text": "http://example.com/goodhint1"}], "ResponsibleOffice": {"Name": [{"Language": "de", "Text": "Bundesamt f\u00fcr Verkehr BAV"}], "OfficeAtWeb": [{"Language": "de", "Text": "https://www.bav.admin.ch/bav/de/home.html"}]}}, {"Type": {"Code": "Hint", "Text": [{"Language": "de", "Text": "Hinweis"}]}, "Index": 3, "Title": [{"Language": "de", "Text": "Guter Ratschlag"}], "Lawstatus": {"Code": "inForce", "Text": [{"Language": "de", "Text": "Rechtskr\u00e4ftig"}]}, "TextAtWeb": [{"Language": "de", "Text": "http://example.com/goodhint2"}], "ResponsibleOffice": {"Name": [{"Language": "de", "Text": "Bundesamt f\u00fcr Verkehr BAV"}], "OfficeAtWeb": [{"Language": "de", "Text": "https://www.bav.admin.ch/bav/de/home.html"}]}}, {"Type": {"Code": "Hint", "Text": [{"Language": "de", "Text": "Hinweis"}]}, "Index": 3, "Title": [{"Language": "de", "Text": "Seht guter Ratschlag mit langem titel: Lorem ipsum dolor sit amet, consectetur adipiscing elit. Cras convallis venenatis posuere. Nunc sollicitudin fermentum bibendum. Nulla ut massa enim. Sed rhoncus euismod venenatis"}], "Lawstatus": {"Code": "inForce", "Text": [{"Language": "de", "Text": "Rechtskr\u00e4ftig"}]}, "TextAtWeb": [{"Language": "de", "Text": "http://example.com/goodhint3"}], "ResponsibleOffice": {"Name": [{"Language": "de", "Text": "Bundesamt f\u00fcr Verkehr BAV"}], "OfficeAtWeb": [{"Language": "de", "Text": "https://www.bav.admin.ch/bav/de/home.html"}]}}]}, {"LegendText": [{"Language": "de", "Text": "Belastet, untersuchungsbed\u00fcrftig"}], "Theme": {"Code": "ch.StatischeWaldgrenzen", "Text": [{"Language": "de", "Text": "Statische Waldgrenzen"}]}, "Lawstatus": {"Code": "inForce", "Text": [{"Language": "de", "Text": "Rechtskr\u00e4ftig"}]}, "ResponsibleOffice": {"Name": [{"Language": "de", "Text": "Bundesamt f\u00fcr Verkehr BAV"}], "OfficeAtWeb": [{"Language": "de", "Text": "https://www.bav.admin.ch/bav/de/home.html"}]}, "Map": {"ReferenceWMS": [{"Language": "de", "Text": "https://wms.geo.admin.ch/?SERVICE=WMS&REQUEST=GetMap&VERSION=1.3.0&STYLES=default&CRS=EPSG%3A2056&BBOX=2608564.1173571427%2C1261670.8114999998%2C2609194.3826428573%2C1262028.7715&WIDTH=493&HEIGHT=280&FORMAT=image%2Fpng&LAYERS=ch.bav.kataster-belasteter-standorte-oev.oereb&SRS=EPSG%3A2056"}], "layerIndex": 1, "layerOpacity": 0.75, "min": {"type": "Point", "coordinates": [2608564.1173571427, 1261670.8114999998], "crs": "EPSG:2056"}, "max": {"type": "Point", "coordinates": [2609194.3826428573, 1262028.7715], "crs": "EPSG:2056"}}, "SymbolRef": "http://localhost:6543/oereb/image/symbol/ch.StatischeWaldgrenzen/1/StaoTyp2.png", "LengthShare": 101, "TypeCode": "StaoTyp2", "TypeCodelist": "https://models.geo.admin.ch/BAFU/KbS_Codetexte_V1_4.xml", "LegalProvisions": [{"Type": {"Code": "Law", "Text": [{"Language": "de", "Text": "Gesetzliche Grundlage"}]}, "Index": 710, "Title": [{"Language": "de", "Text": "Bundesgesetz \u00fcber den Wald"}], "Lawstatus": {"Code": "inForce", "Text": [{"Language": "de", "Text": "Rechtskr\u00e4ftig"}]}, "TextAtWeb": [{"Language": "de", "Text": "https://www.admin.ch/ch/d/sr/c921_0.html"}], "ResponsibleOffice": {"Name": [{"Language": "de", "Text": "Bundeskanzlei"}], "OfficeAtWeb": [{"Language": "de", "Text": "https://www.bk.admin.ch"}], "UID": "CHE263297189"}, "Abbreviation": [{"Language": "de", "Text": "WaG"}], "OfficialNumber": [{"Language": "de", "Text": "SR 921.0"}]}, {"Type": {"Code": "Law", "Text": [{"Language": "de", "Text": "Gesetzliche Grundlage"}]}, "Index": 720, "Title": [{"Language": "de", "Text": "Verordnung \u00fcber den Wald"}], "Lawstatus": {"Code": "inForce", "Text": [{"Language": "de", "Text": "Rechtskr\u00e4ftig"}]}, "TextAtWeb": [{"Language": "de", "Text": "https://www.admin.ch/ch/d/sr/c921_01.html"}], "ResponsibleOffice": {"Name": [{"Language": "de", "Text": "Bundeskanzlei"}], "OfficeAtWeb": [{"Language": "de", "Text": "https://www.bk.admin.ch"}], "UID": "CHE263297189"}, "Abbreviation": [{"Language": "de", "Text": "WaV"}], "OfficialNumber": [{"Language": "de", "Text": "SR 921.01"}]}, {"Type": {"Code": "LegalProvision", "Text": [{"Language": "de", "Text": "Rechtsvorschrift"}]}, "Index": 2, "Title": [{"Language": "de", "Text": "Baugesetz, 3891.100"}], "Lawstatus": {"Code": "inForce", "Text": [{"Language": "de", "Text": "Rechtskr\u00e4ftig"}]}, "TextAtWeb": [{"Language": "de", "Text": "https://oereb-gr-preview.000.ch/api/attachments/197"}], "ResponsibleOffice": {"Name": [{"Language": "de", "Text": "Bundesamt f\u00fcr Verkehr BAV"}], "OfficeAtWeb": [{"Language": "de", "Text": "https://www.bav.admin.ch/bav/de/home.html"}]}}, {"Type": {"Code": "Law", "Text": [{"Language": "de", "Text": "Gesetzliche Grundlage"}]}, "Index": 1, "Title": [{"Language": "de", "Text": "Raumplanungsverordnung f\u00fcr den Kanton Graub\u00fcnden"}], "Lawstatus": {"Code": "inForce", "Text": [{"Language": "de", "Text": "Rechtskr\u00e4ftig"}]}, "TextAtWeb": [{"Language": "de", "Text": "https://www.gr-lex.gr.ch/frontend/versions/pdf_file_with_annex/2934?locale=de"}], "ResponsibleOffice": {"Name": [{"Language": "de", "Text": "Bundesamt f\u00fcr Verkehr BAV"}], "OfficeAtWeb": [{"Language": "de", "Text": "https://www.bav.admin.ch/bav/de/home.html"}]}, "Abbreviation": [{"Language": "de", "Text": "KRVO"}], "OfficialNumber": [{"Language": "de", "Text": "BR 801.110"}]}, {"Type": {"Code": "Law", "Text": [{"Language": "de", "Text": "Gesetzliche Grundlage"}]}, "Index": 1, "Title": [{"Language": "de", "Text": "Bundesgesetz \u00fcber die Raumplanung (Raumplanungsgesetz, RPG)"}], "Lawstatus": {"Code": "inForce", "Text": [{"Language": "de", "Text": "Rechtskr\u00e4ftig"}]}, "TextAtWeb": [{"Language": "de", "Text": "http://www.lexfind.ch/dtah/167348/2"}], "ResponsibleOffice": {"Name": [{"Language": "de", "Text": "Bundesamt f\u00fcr Verkehr BAV"}], "OfficeAtWeb": [{"Language": "de", "Text": "https://www.bav.admin.ch/bav/de/home.html"}]}, "Abbreviation": [{"Language": "de", "Text": "RPG"}], "OfficialNumber": [{"Language": "de", "Text": "SR 700"}]}, {"Type": {"Code": "Law", "Text": [{"Language": "de", "Text": "Gesetzliche Grundlage"}]}, "Index": 1, "Title": [{"Language": "de", "Text": "Raumplanungsgesetz f\u00fcr den Kanton Graub\u00fcnden"}], "Lawstatus": {"Code": "inForce", "Text": [{"Language": "de", "Text": "Rechtskr\u00e4ftig"}]}, "TextAtWeb": [{"Language": "de", "Text": "https://www.gr-lex.gr.ch/frontend/versions/pdf_file_with_annex/2936?locale=de"}], "ResponsibleOffice": {"Name": [{"Language": "de", "Text": "Bundesamt f\u00fcr Verkehr BAV"}], "OfficeAtWeb": [{"Language": "de", "Text": "https://www.bav.admin.ch/bav/de/home.html"}]}, "Abbreviation": [{"Language": "de", "Text": "KRG"}], "OfficialNumber": [{"Language": "de", "Text": "BR 801.100"}]}]}]}, "ConcernedTheme": [{"Code": "ch.Nutzungsplanung", "Text": [{"Language": "de", "Text": "Nutzungsplanung (kantonal/kommunal)"}]}, {"Code": "ch.BelasteteStandorteZivileFlugplaetze", "Text": [{"Language": "de", "Text": "Kataster der belasteten Standorte im Bereich der zivilen Flugpl\u00e4tze"}]}, {"Code": "ch.BelasteteStandorteOeffentlicherVerkehr", "Text": [{"Language": "de", "Text": "Kataster der belasteten Standorte im Bereich des \u00f6ffentlichen Verkehrs"}]}, {"Code": "ch.Grundwasserschutzzonen", "Text": [{"Language": "de", "Text": "Grundwasserschutzzonen"}]}, {"Code": "ch.StatischeWaldgrenzen", "Text": [{"Language": "de", "Text": "Statische Waldgrenzen"}]}], "NotConcernedTheme": [{"Code": "ch.ProjektierungszonenNationalstrassen", "Text": [{"Language": "de", "Text": "Projektierungszonen Nationalstrassen"}]}, {"Code": "ch.ProjektierungszonenEisenbahnanlagen", "Text": [{"Language": "de", "Text": "Projektierungszonen Eisenbahnanlagen"}]}, {"Code": "ch.BaulinienEisenbahnanlagen", "Text": [{"Language": "de", "Text": "Baulinien Eisenbahnanlagen"}]}, {"Code": "ch.ProjektierungszonenFlughafenanlagen", "Text": [{"Language": "de", "Text": "Projektierungszonen Flughafenanlagen"}]}, {"Code": "ch.BaulinienFlughafenanlagen", "Text": [{"Language": "de", "Text": "Baulinien Flughafenanlagen"}]}, {"Code": "ch.Sicherheitszonenplan", "Text": [{"Language": "de", "Text": "Sicherheitszonenplan"}]}, {"Code": "ch.BelasteteStandorte", "Text": [{"Language": "de", "Text": "Kataster der belasteten Standorte"}]}, {"Code": "ch.Grundwasserschutzareale", "Text": [{"Language": "de", "Text": "Grundwasserschutzareale"}]}, {"Code": "ch.Laermempfindlichkeitsstufen", "Text": [{"Language": "de", "Text": "L\u00e4rmempfindlichkeitsstufen (in Nutzungszonen)"}]}, {"Code": "ch.Waldabstandslinien", "Text": [{"Language": "de", "Text": "Waldabstandslinien"}]}], "ThemeWithoutData": [{"Code": "ch.BaulinienNationalstrassen", "Text": [{"Language": "de", "Text": "Baulinien Nationalstrassen"}]}, {"Code": "ch.BelasteteStandorteMilitaer", "Text": [{"Language": "de", "Text": "Kataster der belasteten Standorte im Bereich des Milit\u00e4rs"}]}], "LogoPLRCadastreRef": "http://localhost:6543/oereb/image/logo/oereb/de.png", "FederalLogoRef": "http://localhost:6543/oereb/image/logo/confederation/de.png", "CantonalLogoRef": "http://localhost:6543/oereb/image/logo/canton/de.png", "MunicipalityLogoRef": "http://localhost:6543/oereb/image/logo/municipality/de.png?fosnr=2771", "GeneralInformation": [[{"Language": "de", "Text": "Der Inhalt des \u00d6REB-Katasters wird als bekannt vorausgesetzt. Der Kanton ${canton} ist f\u00fcr die Genauigkeit und Verl\u00e4sslichkeit der gesetzgebenden Dokumente in elektronischer Form nicht haftbar. Der Auszug hat rein informativen Charakter und begr\u00fcndet insbesondere keine Rechte und Pflichten. Massgeblich sind diejenigen Dokumente, welche rechtskr\u00e4ftig verabschiedet oder ver\u00f6ffentlicht worden sind. Weitere Informationen zum \u00d6REB-Kataster finden Sie unter www.cadastre.ch"}]], "Disclaimer": [{"Title": [{"Language": "de", "Text": "Eigentumsbeschr\u00e4nkungen im Grundbuch"}], "Content": [{"Language": "de", "Text": "Zus\u00e4tzlich zu den Angaben in diesem Auszug k\u00f6nnen Eigentumsbeschr\u00e4nkungen auch im Grundbuch angemerkt sein."}]}, {"Title": [{"Language": "de", "Text": "Haftungsausschluss Kataster der belasteten Standorte (KbS)"}], "Content": [{"Language": "de", "Text": "Der Kataster der belasteten Standorte (KbS) wurde anhand der vom Bundesamt f\u00fcr Umwelt BAFU festgelegten Kriterien erstellt und wird fortw\u00e4hrend aufgrund neuer Erkenntnisse (z.B. Untersuchungen) aktualisiert. Die im KbS eingetragenen Fl\u00e4chen k\u00f6nnen vom tats\u00e4chlichen Ausmass der Belastung abweichen. Erscheint ein Grundst\u00fcck nicht im KbS, besteht keine absolute Gew\u00e4hr, dass das Areal frei von jeglichen Abfall- oder Schadstoffbelastungen ist. Bahnbetrieblich, milit\u00e4risch und f\u00fcr die Luftfahrt genutzte Standorte liegen im Zust\u00e4ndigkeitsbereich des Bundes."}]}], "Glossary": [{"Title": [{"Language": "de", "Text": "Baulinien Eisenbahnanlagen"}], "Content": [{"Language": "de", "Text": "Zwischen den Baulinien d\u00fcrfen keine baulichen Ver\u00e4nderungen vorgenommen oder sonstige Massnahmen getroffen werden, die dem Zweck bestehender oder k\u00fcnftiger Eisenbahnanlagen widersprechen."}]}, {"Title": [{"Language": "de", "Text": "Baulinien Flughafenanlagen"}], "Content": [{"Language": "de", "Text": "Zwischen den Baulinien d\u00fcrfen keine baulichen Ver\u00e4nderungen vorgenommen oder sonstige Massnahmen getroffen werden, die dem Zweck bestehender oder k\u00fcnftiger Flughafenanlagen widersprechen."}]}, {"Title": [{"Language": "de", "Text": "Baulinien Nationalstrassen"}], "Content": [{"Language": "de", "Text": "Wenn der projektierte Strassenverlauf definitiv bekannt ist, werden beiderseits der Strasse Baulinien festgelegt. Diese Baulinien erm\u00f6glichen es, die Anforderungen der Verkehrssicherheit und der Wohnhygiene sowie die Erfordernisse eines eventuellen k\u00fcnftigen Ausbaus der Strasse zu ber\u00fccksichtigen. Zwischen den Baulinien d\u00fcrfen ohne Bewilligung weder Neubauten erstellt noch Umbauten bestehender Geb\u00e4ude vorgenommen werden, auch wenn diese nur teilweise in die Baulinien hineinragen."}]}, {"Title": [{"Language": "de", "Text": "Baulinien Starkstromanlagen"}], "Content": [{"Language": "de", "Text": "Zwischen den Baulinien d\u00fcrfen keine baulichen Ver\u00e4nderungen vorgenommen oder sonstige Massnahmen getroffen werden, die dem Zweck bestehender oder k\u00fcnftiger Starkstromanlagen widersprechen."}]}, {"Title": [{"Language": "de", "Text": "BFS-Nr."}], "Content": [{"Language": "de", "Text": "Gemeindenummer aus amtlichem Gemeindeverzeichnis; Eine vom schweizerischen Bundesamt f\u00fcr Statistik mit dem Amtlichen Gemeindeverzeichnis im Jahre 1986 erstmals vergebene Zahl, die der eineindeutigen Bezeichnung von territorialen Einheiten im Einzugsbereich der Schweiz dient."}]}, {"Title": [{"Language": "de", "Text": "E-GRID"}], "Content": [{"Language": "de", "Text": "Eidgen\u00f6ssischer Grundst\u00fccksidentifikator; Aus einem Pr\u00e4fix und einer Nummer bestehende Bezeichnung, die es erlaubt, jedes in das Grundbuch aufgenommene Grundst\u00fcck landesweit eindeutig zu identifizieren, und die zum Datenaustausch zwischen Informatiksystemen dient."}]}, {"Title": [{"Language": "de", "Text": "Eigentumsbeschr\u00e4nkung"}], "Content": [{"Language": "de", "Text": "Der Zweck aller Eigentumsbeschr\u00e4nkungen ist die Wahrung von Interessen anderer Personen, denen jene des Eigent\u00fcmers sich in bestimmter Beziehung unterordnen m\u00fcssen. Dies sind entweder die Eigent\u00fcmer angrenzender Grundst\u00fccke, die Nachbarn, oder auch weitere Kreise von Privatpersonen oder endlich die Allgemeinheit selber, der Staat. Die Beschr\u00e4nkungen zugunsten der Nachbarn oder weiterer Privatpersonen sind regelm\u00e4ssig privatrechtlicher, jene zugunsten der Allgemeinheit \u00f6ffentlich-rechtlicher Natur."}]}, {"Title": [{"Language": "de", "Text": "Gesetzliche Grundlage"}], "Content": [{"Language": "de", "Text": "Gesetz, Verordnung etc., das generell-abstrakt ist (generell f\u00fcr die Person, die nicht bekannt ist, abstrakt, weil der Perimeter ohne Karte definiert ist) und auf Bundesebene, auf kantonaler oder kommunaler Ebene erlassen worden ist und die bloss eine allgemeine Rechtsgrundlage der Eigentumsbeschr\u00e4nkung darstellen. Die gesetzliche Grundlage ist nicht Teil des \u00d6REB-Katasters. Der \u00d6REB-Kataster enth\u00e4lt aber Hinweise auf die entsprechende gesetzliche Grundlage."}]}, {"Title": [{"Language": "de", "Text": "Gew\u00e4sserraum"}], "Content": [{"Language": "de", "Text": "Fliessgew\u00e4sser k\u00f6nnen nur wieder naturn\u00e4her werden, wenn ausreichend Raum in den Schutz der Gew\u00e4sser miteinbezogen wird. Der Gew\u00e4sserraum soll weitgehend frei von neuen Anlagen bleiben; bestehende Anlagen haben jedoch Bestandesgarantie."}]}, {"Title": [{"Language": "de", "Text": "Grundwasserschutzareale"}], "Content": [{"Language": "de", "Text": "Von Kanton festgelegte Areale, die f\u00fcr die k\u00fcnftige Nutzung und Anreicherung von Grundwasservorkommen von Bedeutung sind und in dem keine Bauten und Anlagen erstellt oder Arbeiten ausgef\u00fchrt werden, die k\u00fcnftige Nutzungs- und Anreicherungsanlagen beeintr\u00e4chtigen k\u00f6nnten."}]}, {"Title": [{"Language": "de", "Text": "Grundwasserschutzzonen"}], "Content": [{"Language": "de", "Text": "Grundwasserschutzzonen, in denen gew\u00e4hrleistet werden soll, dass bei unmittelbar drohenden Gefahren wie Unf\u00e4llen mit wassergef\u00e4hrdenden Stoffen ausreichend Zeit und Raum f\u00fcr die erforderlichen Massnahmen zur Verf\u00fcgung stehen."}]}, {"Title": [{"Language": "de", "Text": "Kataster der belasteten Standorte"}], "Content": [{"Language": "de", "Text": "Der Kataster enth\u00e4lt Deponien und andere durch Abf\u00e4lle belastete Standorte, die saniert werden, wenn sie zu sch\u00e4dlichen oder l\u00e4stigen Einwirkungen f\u00fchren oder die konkrete Gefahr besteht, dass solche Einwirkungen entstehen. Die Beh\u00f6rde ermittelt die belasteten Standorte, indem sie vorhandene Angaben wie Karten, Verzeichnisse und Meldungen auswertet. Sie tr\u00e4gt diejenigen Standorte, deren Belastung erwiesen oder sehr wahrscheinlich ist, in den Kataster ein."}]}, {"Title": [{"Language": "de", "Text": "Katasterverantwortliche Stelle"}], "Content": [{"Language": "de", "Text": "Die katasterverantwortliche Stelle im Kanton erh\u00e4lt die in den \u00d6REB-Kataster aufzunehmenden Daten von den zust\u00e4ndigen Fachstellen. Sie verwaltet diese Daten und stellt sie via kantonales \u00d6REB-Geoportal der \u00d6ffentlichkeit zur Verf\u00fcgung."}]}, {"Title": [{"Language": "de", "Text": "KbS"}], "Content": [{"Language": "de", "Text": "Kataster der belasteten Standorte"}]}, {"Title": [{"Language": "de", "Text": "L\u00e4rmempfindlichkeitsstufen"}], "Content": [{"Language": "de", "Text": "Empfindlichkeitsstufen werden festgelegt, um jeweils bestimmte Zonen zu definieren: diejenigen, die eines erh\u00f6hten L\u00e4rmschutzes bed\u00fcrfen, diejenigen, in denen keine st\u00f6renden Betriebe zugelassen sind, und diejenigen, in denen m\u00e4ssig und stark st\u00f6rende Betriebe zugelassen sind."}]}, {"Title": [{"Language": "de", "Text": "Nutzungsplanung"}], "Content": [{"Language": "de", "Text": "Festlegung der Verwendung einzelner Bodenfl\u00e4chen f\u00fcr bestimmte Zwecke (z. B. Landwirtschaft, Siedlung, Wald)."}]}, {"Title": [{"Language": "de", "Text": "\u00d6REB"}], "Content": [{"Language": "de", "Text": "\u00d6ffentlich-rechtliche Eigentumsbeschr\u00e4nkung"}]}, {"Title": [{"Language": "de", "Text": "\u00d6REB-Kataster"}], "Content": [{"Language": "de", "Text": "Kataster der \u00f6ffentlich-rechtlichen Eigentumsbeschr\u00e4nkungen"}]}, {"Title": [{"Language": "de", "Text": "Planungszonen"}], "Content": [{"Language": "de", "Text": "Zone, die ein genau bezeichnetes Gebiet umfasst, f\u00fcr das ein Nutzungsplan angepasst werden muss oder noch keiner vorliegt, und innerhalb derer nichts unternommen werden darf, was die Nutzungsplanung erschwert."}]}, {"Title": [{"Language": "de", "Text": "Projektierungszonen Eisenbahnanlagen"}], "Content": [{"Language": "de", "Text": "Um die freie Verf\u00fcgbarkeit der f\u00fcr k\u00fcnftige Bahnbauten und -anlagen erforderlichen Grundst\u00fccke zu gew\u00e4hrleisten, k\u00f6nnen f\u00fcr genau bezeichnete Gebiete Projektierungszonen festgelegt werden. In diesen Zonen d\u00fcrfen keine baulichen Ver\u00e4nderungen vorgenommen werden, die dem Zweck der Eisenbahnen widersprechen."}]}, {"Title": [{"Language": "de", "Text": "Projektierungszonen Flughafenanlagen"}], "Content": [{"Language": "de", "Text": "Um die freie Verf\u00fcgbarkeit der f\u00fcr Flughafenanlagen erforderlichen Grundst\u00fccke zu gew\u00e4hrleisten, k\u00f6nnen f\u00fcr genau bezeichnete Gebiete Projektierungszonen festgelegt werden. In diesen Zonen d\u00fcrfen keine baulichen Ver\u00e4nderungen vorgenommen werden, die dem Zweck der Flughafenanlagen widersprechen."}]}, {"Title": [{"Language": "de", "Text": "Projektierungszonen Leitungen mit einer Nennspannung von 220 kV oder h\u00f6her"}], "Content": [{"Language": "de", "Text": "Um die freie Verf\u00fcgbarkeit der f\u00fcr Flughafenanlagen erforderlichen Grundst\u00fccke zu gew\u00e4hrleisten, k\u00f6nnen f\u00fcr genau bezeichnete Gebiete Projektierungszonen festgelegt werden. In diesen Zonen d\u00fcrfen keine baulichen Ver\u00e4nderungen vorgenommen werden, die dem Zweck der Flughafenanlagen widersprechen."}]}, {"Title": [{"Language": "de", "Text": "Projektierungszonen Nationalstrassen"}], "Content": [{"Language": "de", "Text": "Um die freie Verf\u00fcgbarkeit des f\u00fcr den Bau der Nationalstrassen erforderlichen Landes zu gew\u00e4hrleisten, k\u00f6nnen f\u00fcr genau bezeichnete Gebiete Projektierungszonen festgelegt werden. In diesen Zonen d\u00fcrfen keine baulichen Ver\u00e4nderungen vorgenommen werden, die dem Zweck der Nationalstrassen widersprechen."}]}, {"Title": [{"Language": "de", "Text": "Rechtsvorschrift"}], "Content": [{"Language": "de", "Text": "Generell-konkrete Rechtsnorm, die zusammen mit den ihr zugeschriebenen Geobasisdaten die Eigentumsbeschr\u00e4nkung unmittelbar umschreibt und innerhalb desselben Verfahrens verabschiedet worden ist."}]}, {"Title": [{"Language": "de", "Text": "Sicherheitszonenplan"}], "Content": [{"Language": "de", "Text": "Zonenplan, in dem die Sicherheitszonen dargestellt sind und aus dem die Eigentumsbeschr\u00e4nkungen nach Fl\u00e4che und H\u00f6he ersichtlich sind."}]}, {"Title": [{"Language": "de", "Text": "Statische Waldgrenzen"}], "Content": [{"Language": "de", "Text": "Statische Waldgrenzen m\u00fcssen auf der Grundlage rechtskr\u00e4ftiger Waldfeststellungen festgelegt werden. Neue Bestockungen ausserhalb dieser Waldgrenzen gelten nicht als Wald."}]}, {"Title": [{"Language": "de", "Text": "Vorwirkung"}], "Content": [{"Language": "de", "Text": "Mit der \u00f6ffentlichen Auflage entsteht eine Vorwirkung: Ab diesem Zeitpunkt d\u00fcrfen Bauvorhaben in der Regel nur bewilligt werden, wenn sie der \u00f6ffentlich aufgelegten Planung nicht widersprechen."}]}, {"Title": [{"Language": "de", "Text": "Waldabstandslinien"}], "Content": [{"Language": "de", "Text": "Bauten und Anlagen in Waldesn\u00e4he sind nur zul\u00e4ssig, wenn sie die Erhaltung, Pflege und Nutzung des Waldes nicht beeintr\u00e4chtigen."}]}, {"Title": [{"Language": "de", "Text": "Waldreservate"}], "Content": [{"Language": "de", "Text": "Gesch\u00fctzte Waldfl\u00e4che, die der Erhaltung der Artenvielfalt von Fauna und Flora dient."}]}, {"Title": [{"Language": "de", "Text": "Zust\u00e4ndige Stelle"}], "Content": [{"Language": "de", "Text": "Durch die Gesetzgebung bezeichnete Stelle des Bundes, des Kantons oder der Gemeinde, die f\u00fcr das Erheben, Nachf\u00fchren und Verwalten der Geobasisdaten zust\u00e4ndig ist."}]}]}}}
```

Reply:
Should be fixed now.